inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1113,7 +1113,9 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
         auto& detection_output_node = node.as<detection_output>();
         auto confidence_layout = detection_output_node.confidence().get_output_layout();
         auto prim = detection_output_node.get_primitive();
-        if (confidence_layout.size.batch[0] <= lws_max && confidence_layout.size.batch[0] >= 4 && prim->confidence_threshold >= 0.1 &&
+        auto batch_size_limitations = (device_info.supports_immad && device_info.execution_units_count >= 256) ? true : confidence_layout.size.batch[0] >= 4;
```

Comment:
So in the DG2, even though bsize < 4 gpu is faster?

Reply:
Yes, in all cases GPU implementation is faster (in case of >= 256EU count). I think it may perform faster even on smaller EUs count, but it looks like such GPUs haven't enough memory. Maybe later we can change this logic and make it more flexible in the future to cover more cases

Reply:
Yes, because of host to device transfer
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABMwAAAESCAYAAAABuEmbAAAgAElEQVR4Aey9/6seSX7fW//FyKwG/+RMGO0unsmR0ZwZE4wzO8mMJtw1mSBPzspflD1E1u7Fs8vaOONd52QmXsd3sjk25IeNkBGaI8sRssxB5CAQmpWFD/aAA7lGQ5aICxcuHLwY5ycbBrzhc6mq/nR/6ls//TxPVT/95X3g0N1V3fXlXdWfqno9VdXq5OSE1v3/7ne/S1P8e/LkyRSzhTxBAaMA6ndYEaBJqEluF2gcKgpNQk1yu0DjUNHf+73fCx3hklUB1LtQTmgSapLbBRqHikKTUJPcLtA4VBSahJrkdulDY7UuLNPPA5jlLnqEBwXKK9CHgSmfi7wxQJO8esZCg8ahKtAk1CS3CzQOFQUwCzXJ7YJ6FyoKTUJNcrtA41BRaBJqktsFGoeKQpNQk9wufWgMYNZSan0UQEv08IICRRVA/Q7lhSahJrldoHGoKDQJNcntAo1DRQHMQk1yu6DehYpCk1CT3C7QOFQUmoSa5HaBxqGi0CTUJLdLHxoDmLWUWh8F0BI9vKBAUQVQv0N5oUmoSW4XaBwqCk1CTXK7QOPciiK8Lgqg3oUqQZNQk9wu0DhUFJqEmuR2gcahotAk1CS3Sx8aA5i1lFofBdASPbygQFEFUL9DeaFJqEluF2gcKgpNQk1yu0Dj3IoivC4KoN6FKkGTUJPcLtA4VBSahJrkdoHGoaLQJNQkt0sfGquTgx1SStn/rT061h8BkG7spxRt7R1HPxCAPcxyFz3CgwLlFejDwJTPRd4YoElePWOhQeNQFWgSapLbBRrnVhThdVEA9S5UCZqEmuR2gcahotAk1CS3CzQOFYUmoSa5XfrQWCmGZG1fyzzeoy21QweJewDMchc9woMC5RXow8CUz0XeGKBJXj1joUHjUBVoEmqS2wUah4piD7NQk9wuqHehotAk1CS3CzQOFYUmoSa5XaBxqCg0CTXJ7dKHxioFweTXMw92FO0cnERnl+ErmbmLHeFBgX4U6MPA9JOTfLFAk3xapkKCxqEy0CTUJLcLNA4VBTALNcntgnoXKgpNQk1yu0DjUFFoEmqS2wUah4pCk1CT3C59aKz2tqrlmCoBxfTssgWz0DDDLHfRIzwoUF6BPgxM+VzkjQGa5NUzFho0DlWBJqEmuV2gcagogFmoSW4X1LtQUWgSapLbBRqHikKTUJPcLtA4VBSahJrkdulDY9XMHDugHbVFe8fuTDI9uyy1dxnPQgMwy130CA8KlFegDwNTPhd5Y4AmefWMhQaNQ1WgSahJbhdoHCoKYBZqktsF9S5UFJqEmuR2gcahotAk1CS3CzQOFYUmoSa5XfrQWJlN/qu9yUI4piFaeu8yALPcRY7woEB/CvRhYPrLTZ6YoEkeHdtCgcahOtAk1CS3CzQOFQUwCzXJ7YJ6FyoKTUJNcrtA41BRaBJqktsFGoeKQpNQk9wufWgcALNmxln1tcydg+TeZQBmuYsc4UGB/hTow8D0l5s8MUGTPDq2hQKNQ3WgSahJbhdonFtRhNdFAdS7UCVoEmqS2wUah4pCk1CT3C7QOFQUmoSa5HbpQ+NmSWbkS5iLNvsHMMtd5AgPCvSnQB8Gpr/c5IkJmuTRsS0UaByqA01CTXK7QOPciiK8Lgqg3oUqQZNQk9wu0DhUFJqEmuR2gcahotAk1CS3Sx8aqx3Fm/77+5fF9zRjSCaP2MMsd9EjPChQXoE+DEz5XOSNAZrk1TMWGjQOVYEmoSa5XaBxbkURXhcFUO9ClaBJqEluF2gcKgpNQk1yu0DjUFFoEmqS26UPjZUEX6ueA5jlLnqEBwXKK9CHgSmfi7wxQJO8esZCg8ahKtAk1CS3CzQOFcUeZqEmuV1Q70JFoUmoSW4XaBwqCk1CTXK7QONQUWgSapLbpQ+NAcxaSq2PAmiJHl5QoKgCqN+hvNAk1CS3CzQOFYUmoSa5XaBxqCiAWahJbhfUu1BRaBJqktsFGoeKQpNQk9wu0DhUFJqEmuR26UNjALOWUuujAFqihxcUKKoA6ncoLzQJNcntAo1DRaFJqEluF2gcKgpgFmqS2wX1LlQUmoSa5HaBxqGi0CTUJLcLNA4VhSahJrld+tAYwKyl1PoogJbo4QUFiiqA+h3KC01CTXK7QONQUWgSapLbBRqHigKYhZrkdkG9CxWFJqEmuV2gcagoNAk1ye0CjUNFoUmoSW6XPjQGMEuUmqo/hsAfRcARmqAOoA6gDqAOoA6gDkyjDmhghrKcRlmiHFGOqAOoA6gDqANzrQMJnJPNWT1+/JgePXpEDx8+pAcPHtD9+/fp6OiI7t27R4eHh3T37l26c+cO3b59m27dukU3b96kGzdu0PXr1+natWt09epV0pv+a7o3pf+5VjjkG8YWdQB1AHUAdQB1YPp1QPckUc7TL2eUMcoYdQB1AHUAdWDKdaA0g8IMswR75EqV8IYzFBi1Aqjfoy4+JB4KQIElFIC9W0Is3AoFoMCoFYC9G3XxIfFQAAosoUBf9g7ALFEofRVAIno4Q4GiCqB+F5UXgUMBKDAgBWDvBlQYSAoUgAJFFYC9KyovAocCUGBACvRl7wDMEoXeVwEkooczFCiqAOp3UXkROBSAAgNSAPYuXhjY9D+uC1yhwJgVgL0bc+kh7VAACiyjQF/2DsAsUSp9FUAiejhDgaIKoH4XlReBQwEoMCAFYO/ihQFgFtcFrlBgzArA3o259JB2KAAFllGgL3sHYJYolb4KIBE9nKFAUQVQv4vKi8ChABQYkAKwd/HCADCL6wJXKDBmBWDvxlx6SDsUgALLKNCXvVMnBzvNV5K29uj45IRO9P/xHm0p/qLEDh2we+Sov5I5tb++CmBquiE/41AA9Xsc5dSeyqe0v802ujnuHkWeerpP27U9t/fK+57ubzftgFIk/SKhVU6x+HfJj57D3t5/mg4q4pN6jt25DjtpPdp18hGPM5ZuRfZe6+eEqdOmw93ep+VyEMkUnDaiANeVjUQ+4EjHBczi723wrmq9i9i7qiD9sD27wPYpbnvilYGf4Xra5CmS58bTBOY+u02hmY2EoWDv4iUxDVeuR9PIzVxzEX9vvdffiuPbJK8P59qIjP07L95o2oLii+TLe5DT69tQduf67T1WxRQJH/YuKIUpOXB9KJ0npSQkq2HYMe1tKdo5qOCZhmrR+6w/gFnpYkL4UCCvAn0ZmLypRmiuArZj4HQqKmAUc3M7F0e0qyq4pTs9YtBnOyUh+HLj1ld+/FVHpY5Ix6EHZkcG7DlpCgMTLi3PtaZVPycGjKYzJ67rGGw662TW7vok4Qdg5qg0tgvYu3iJjRGYOXakV3tXgXNpYwyb265ge4vdistvXVttWsIeVeEZWy1sdzyatjASfrB3cSlH4gp7N5KCak2mfTc3Z+/8+O21qjtObO+qnxGNLY71t/xMJmyOuY3DjPQZW+2kjKMt/IQf7J0UcHTnfdk7FZ85dkA7Ss4q868rkFYBNgCz0dUvJHjmCvRlYGYuc+Hs28bf6VDpGE3HhYGX7YDUfZwuKUqCJv/hMP74AC68zw8pft3hOZlWc8751iHqvMc6cDbcuCYJP3So4kU0ElfYu5EUVGsyE/agN3vX1ZYm0tmaN+EpbVoK4JvbU/ZNhGVOEzatzQ/2zhdxVNewd6MqrkRiE3akN3sXxu/075x06Cy02RmZxS73hXHLEMy5Yyelb1v4CT/YOyng6M77sndKzyTjyOoZZSeRGWY7B3apZj0LrYFmAGajq19I8MwV4Hd+5jKMPPupToUY2AWdmg5Z7vyMH79/zXGl3Nk/dezwnJfWo13dnmlo1vas9QMwS+k+PXfYuymUaeqd7sneBUA+pWkqnan7PXfHptmwuP4q+QNANWDclsvyo0YN9s5TePKXXF8mn9FJZzBlR3qyd0Efyk2PA89MObj+6aKx93EddWxa/VCHsBw7WT+4ANzZcAMzqcNaOFNXxoHzISnAdal0mlQDyfQssi3aO2YQZqGZSUjLcky93xmAWeliQvhQIK8CfRmYvKlGaK4CqU5FukNlOjn1XmZyNhaHnAqT/eXR3st1yR7TM7qCmXAyqOj5orTE/S00Uy0doFi6WQvrhw5VtEBG68h1dLQZQMLrgVBoR3qyd8kBml84cbvk3xW/bn/W2u/KVpn0CHtrAFpsfyLYu7jW03WFvZtC2aZsQU/2rgJmXJf8/l0IzPTiBt4bsbv+jk2rH0vlnW9o84e9Y5XmcuQ6Wjq/qt7k/+SEDnYUbe0dVxv+C3hmPgAgl2gyVLNHALPSxYTwoUBeBfoyMHlTjdBcBVKdhnSHqn4+MfgzsKnzL22R+P1BnIkwcl+dkLaT9ufCtIp8V3sLtf16GUAxkdbAT+ersy5teYLfJhSAvYurPvo9zEy2xHufsGvuMvVGi9CGNH7BWSrs4MZ2uxXcLhwWp6ctr6l4rXtg00y8CT/YO1Eq4zuFvRtfmYUpTr3PbTagCiVhqxbbF5mKSPwmXAvpQ2AWuV8GlzwX+anvaQ+rPR/2Wdi7WszJn/Rl7wJgpmecHe9tWXAmll9qmNbMRgMwm3wNRAYnrUBfBmbSIm48c4lOhdNZinVG/H3ObEbaOyGxzMbij8UXuy8Wnu+Wfi6W1u4dOBtuW4fKn8UShu2nFddDVgD2Ll46kwBmvdm7mG2L6Zq2W7G72S1m09ivPsp9e+S5uSEVL+xdrd9MTmDvplDQife5N3sXi1/YQCcdWu82O9NSHoEda8Ly+2HaZ7GdbEtHLE/6o8rb+EG0pYiG7tWXvWuWZMpZZPLcQDN/uSaA2dArENIHBdoU6MvAtKUBfusqEGn8q2U5sqNhOgPeZ8bdGRc2nNQMKtNBidKlSPymEyWWCZksRu7jjk80XNYl9lxLWv0OXKVFGIUNI3S38Vq9eImmdhOdRE4ajqNSAPYuXlyjB2a92rtqYCX3Eatmskp7ywNH160a6EWNTtqmHe26ttQdLHr2MWp7dbnD3sVr/3RdYe+mULbe+21e5X3aNl8er75MWc+k95ZiO32htH3RQa7ev/P6RU6c6XDbbRqXWyTvvER04Ux/2DtWcS7Hvuyd2qn3sxFLMDUkO9ipPwagE5OaXYY9zOZSJZHPKSnQl4GZkmbDy0vVEaptuN7w3h1g1WmuBpZc7vpYD+hMR6f5+Et9TzW4W9Shqu+v0tGMCW2HyvfneNPhtjzXJa1CD46r1sGcxHQTeohOKKc9Ho4bKq6GqwCX43BTuJmUjRGYcVnaY5/2zpYR/wBRp6MewLXYrbaBaZtN8/3quKr64tn1xvbK+gR7J9WYwznXzTnkdbp5jL23fdq7WPwemHPsj5u2ZP+u1aa12FD/Oe7nBUYvnm7Zh/NtuPSbbn2abs76sndKA691/7GH2XQrYtGcaQPodwBXitAayMBurhTWPB7qy8DMQ80J59J0iORsq0x5LRVupuQhmGkpAHsXL89xAbN4HrK6lrJLpcLNmnkENhUFYO+mUpKF81HKLpUKt7AcCH6cCvRl7wDMEvWjrwJIRD8P556BGf+qsMyvCeZXkiWgnonDJ3feLyPLxF+qIqB+l1J2WuHq+lyivpYKd1rqIze5FIC9y6XktMMpZZdKhTvt0kDuVlUA9m5V5eb1XCm7VCrceZUOcttVgb7sHYBZokT6KoBE9PNw7g2Y2Wm+2/tHtL/tLr3KLbRuKJQDzHTcYqqy+eVFXOdOQMfwUL87CoXboAAUGL0CsHejL0JkAApAgY4KwN51FAq3QQEoMHoF+rJ3AGaJqtJXASSid5wNhOH12hV8CdeH22WJdjYIL1GU68E9SGPATbNvkcN4TOw6DP2MDEMvzbJhW328MB0/Hba3lMuJc5v29/0lmTLsyPOOKvKiQ37rPC0HzFyd2+Lx017FE0xN9gCazEaP50Oq3z1mG1FBASgwQwVg72ZY6MgyFJipArB3My14ZBsKzFCBvuydevz4MT169IgePnxIDx48oPv379PR0RHdu3ePDg8P6e7du3Tnzh26ffs23bp1i27evEk3btyg69ev07Vr1+jq1auk9zB78uTJpP77KoCFdTsALtUTZpmfAFLmPgZYDG/42v9iiYVgzVKrI9oNlh1yGBwHXzebPhqQVz9X+QvyZkGf+7wTp4aA9fN+Gpf51C+nrcmvGzerbO9r0sDu6WMMmMmN1V3/Ks1CAx2yucfAw+XjT6dsPR+u31N7b5GfadlhlCfKM0cdgL2L1yO9h1kOfRFGXF/oAl02UQdg71DvNlHvECfq3SbqQF/2DjPMEtyBCyDh3Z9zNSvLYzAaw9Cu8uBVfZMFM/WlTq1c/ujDtmhuwjAMhJKBynCiYE/Mpor5yzQ5wI8TJJ5np+gxTKvVpwFo9rHlgZULxCLxyDzw1/WkRlV6LTRzAWE0Kz05DqZ+95TfrtFoY4+/sgpA41BfaBJqktMF9i6uJjb9j+uS0xXvdqgmNAk1yekCexdXE/UurktOV2gcqglNQk1yuvRl7wDMEqXWVwEkonedK2im0yRnRzUAywc5/nUEmImZXW5kfBWG0cRX3SOBmTznIKolmoYfxfwlbBJ5ZO3t0YdedeDiJEyrDxTtzfY+qaEIJHq6PjCLgE25p1k01vKOrHH5mMYVAxq28uUFjUONoUmoSU4X2Lu4mgBmcV1yuuLdDtWEJqEmOV1g7+Jqot7FdcnpCo1DNaFJqElOl77sHYBZotT6KoBE9AlnF74Qz9riY/1UBCBJOBWDV/WzfBKG0QrMgjTocMQMsZi/TFPMn5Oy8Bim1Ym7fr5/YGY0c+Dk8mmok5/xZJj1O2MGVwwKDduKwi3xGDQOxYImoSY5XWDv4moCmMV1yemKdztUE5qEmuR0gb2Lq4l6F9clpys0DtWEJqEmOV36sncAZolS66sAEtEnnH0wZK9NWp1lgP593gyzajlnM9MqvYeZDLYVmPGG/+IBFxZZ2FfHaQCZXKK4GCS5s72kRKxDMxvNjZvvjceRDtffV22RrpF913w4WeVbyMSJ6/U4zPrdqwTRyNCwRWXJ6giNQzmhSahJThfYu7iaAGZxXXK64t0O1YQmoSY5XWDv4mqi3sV1yekKjUM1oUmoSU6XvuwdgFmi1PoqgET0jbMBLs3XLJVHWgwYEnuZ2QcXgx07O43DbUBTE3EYRjsw009aKMbayQ39TbgMycwXP3fpSF9HZl/Vzy/4KICb1m3aP9qn7fprovyxgUi6qnsY3uUEZlJXJ/w6Xe6y2iYP/Z6xxv3Guii2sM4teiK3/ygaNjkzM7cAPYQ3LI03X+e05MPSJFIJRl7nYO8iZToWp5HXvWG927B3nar9yOsc7F28lIf1LsbT6Ow3nbhlyM7D0hj2rlNdgb3rJJM6OdghNq5qa4+OT07oRP8f79FWPcjfoQN2jxz1VzKn9seaDD5fuqKbLzAOPqXrJdDANgnB1guufrpUuHUEwzxJ1m8HalqgKhmtgYu1XdD+skwSjVNnY5x4PoOEAexNhBlt7DeqSSShnfWMPOs5mfJ0oLV3Q4HLThqLNE29zmmJh6VJpNBHXudg7yJlmnKCvUsps5L7sN7tzbexWsRhaRIpVti7alyI/l3Z/sf06t6w3m3Yu0gNC52mau/CnK7lohxIVsOwA9pRW7R3XMEzDdUkTKvvs/4AZmuVwVoPG2MuicZaoQ33YQ08eMZWzlSWCjdnGkuEFR1AaqMZzFbUswabTpOub7IcDIiq/RONU2djnHg+gwArA7ONaxLJfGc9I88OwCnoUBmN3Vmu8r2cep3TRTIsTSKVZOR1DvYuUqYxJ9i7mCpruQ3r3d58G6vFHJYmkeKFvTOioH9nt2Up1+edXt0b1rsNexepYaHTFO1dmMu1XVR05pgGZDsHdqaZgWPHtLclABqA2drC5wnALoGcAS/LIxdCqRUIB5Dd6pIPL3gZrq2DicapszHm5+XSXhekEO+VV89ya2Cezpzt4MmlxjZMzq8+ys5PLUh14jb2Q9DEZEosNd6m/f1dcpc7+3lsNAmBur2XNQj8vdkljW1Jx+FruOh6WY2nXue0XsPSZHp1jt//pm4O4d2279Tu0ebsnbuH2RA0mV7dG9a7vfk6B3sntw5RVKKNhb1rLL08W/Zd1M+W7X/A3pXVF/ZO1v/63Onnj39MEdq7OqdZT9TeFg8uFe0ciBllDjA7oYMd4Q9glrUQEBgU6FuBwMCYmQUNaEmlJ2jcjOFlqMWNk/f0ksBMKQ4v/tEFuY+f8wuoSUs8D+a+pmfqJbC5dDpUA9KEARcDSgnMdJkEmvCSRj8PTnn5+toBsxNXFU5rHI18nc4cjVvKjAObep3T+RyiJk490ICa61Q1iBhTnYO947fJPTrAzLcV7q31VR/v45Tq3hDf7U22sboiDVGTKdU52LvaXDknTr2DvaNd9O9ofzvSf+ZVMy39065jCtg7OUmh+cBgzjFFYO+ctz7fhaoh2YlYhmn2LxMzyqp9zpp7K7BWgTMsycxXIAgJCvShQGBgvM6DO1OrgVD+YMkYvXognQeYOVxLp4vDjzZeGvJUgM34y19sGyW7Nm5tHaqNaBLLc6BJAxhtjoUm1Yc4WFOTB9bTBx9eHajVM2loi6O+s9NJm8axAKZe53SeB6XJBOsc7F3szSJqA2awd1IzaVOl++LzQb3b1Qxtbg9M6oP2pGnvbe5E3jO0sTrMQWkCe0dHVTWeelvbVu9g76pKYA7inZfOHc7bNI49PvU6p/M8KE3mYO9iFS2Dm6o3+T+xs8i29o7tUkzvYwB7OwKgYYZZBukRBBTYnAKLBpB1yjyIYgBZvRzSnXXCyyWdzrgOSHbI64BjJxHgJp/10mJD8J6pOvQ6f80vxtVSzSBhYRo6NWxeOopq4sVlUiw1EfnlMrXHBnCZ9Jm8e1rFgJmAabU6HeKo7+1w0kljEU5RfRcNIGP6+88IfVapczqrg9IklueR1zl+N+pqFcuj9vTcp1732oBZSquimnj6mzSMvO4N6t32bRfXebb7Mf39Z2Dvqg3xh9vGwt7V1ss56fQueu8A7J0j4cKLThqLUIrq69suHa9sT7yytsny+smwd+Ozd6J+5TwNgFl8FpmefZb+UiZmmOUsEoQFBcorEHSovJlIdQq8BkU3bhIK1PeZE9vQ+P7+rCb3GXnlNVTaSzZupuFq+fVbBuXlx6RhWWDmhVEH36cmsTwv1KROqT3hMPgovE1nhXXx8lXfFnmu9lvhxOlQpTQW4U69zumsDkqTWHmPvM7B3okXSpw6wCz1Lnp2oej7OMG6N6h3e9EAMqa/qRcNHGqqj56F0szo7trGwt75fRjexyri3oi91BnsXVyuTu8i7F1cvI6unTQWYRVtT2DvhNLVaczGT65/F2Y7h0uzJNMsw4xDMb1/WT3zzJtddnJyQgBmOYoCYUCB/hQIO1TNhvnMT0xqluo8cBiy4+d2qnWYDqRxsrwAmFWNX3LvpJawukI7t7Hn/DSDgv41sfrVENI0dnJmXxxSOlJUum1vbzt7nel73LLw4tIDJTPzoEscboxtV3GN3QGZLi/Oc3uHistovHVOazUsTbx6MIE6B3vX9kY2fsZOCghifHptA6ZX94b1bm++jYW9kz86lmljYe8amybP4u8i+nd6Hy/ub0m9VjmPa4z+ndTStrOb0mR6bWzM3km9c52rnXp5lVxyqb+K2XwMoA2WAZjlKgqEAwX6UyBpYHhwXNsFtyFdBC90DnjQxXH4DbELaWSeF3Xm9b3W2HPY9f5mxmu3mjpc2S5J/kS+/PTIFPiNvfETz3K8MoyymvCvz2yPd+lIp4eX0NgE2o1LRZk5uogykZJYydwNT8nJq2zQbdlw/s3RSYNUsf08prFfZ2T6y+q7+Tqn1RqWJtOrc1xvg5rp1Hf7jvX3bg+j7g1Lk+nVvWG928Ooc8PSZHp1DvYusGrGIVbv3D5P323A9OpeTGP0754EFXJzmkyvziXtXaD6eg5KA691/zHDbL1CwNNQoG8F+jIwQb7MAFXOBgru2KhDrLEvnqCBa5I7/71rPAJ9oUnuWuaGB3vn6rHRqxG8jzn1wbsdqglNQk1yusDexdXsvd7pZMDexQsjl+sI9O293o1Ak1zFr8Ppy94BmCVKra8CSEQPZyhQVIFN1W/9q4qcwVE0kysE3nvDZvpTw9ZkBRlbH+lb46HXOS0WNGmtMmt7wt7FJXT3MIvfk9t1DO9jzjzj3Q7VhCahJjldYO/iavZd73QqYO/iZZHLdQz69l3vxqBJrvLX4fRl7wDMEqXWVwEkooczFCiqAOp3XN6+G7Z4KqbtCo3D8oUmoSY5XWDv4mpuApjFUzJdV7zbYdlCk1CTnC6wd3E1Ue/iuuR0hcahmtAk1CSnS1/2DsAsUWp9FUAiejhDgaIKoH7H5UXDFtclpys0DtWEJqEmOV1g7+JqApjFdcnpinc7VBOahJrkdIG9i6uJehfXJacrNA7VhCahJjld+rJ32YAZJxhH3pwbR9QF1AHUAdQB1AHUAdSBYdYBDcxQNsMsG5QLygV1AHUAdQB1AHWgWx3ICeFiYanHjx/To0eP6OHDh/TgwQO6f/8+HR0d0b179+jw8JDu3r1Ld+7codu3b9OtW7fo5s2bdOPGDbp+/Tpdu3aNrl69SnrTfxRotwKFTtAJdQB1AHUAdQB1AHVg03UAwAx1cNN1EPGjDqIOoA6gDqAOrFsH9Ey+kv/ZZpjFaNyY3bjgxpwHpB0KpBTg+v3p5SuEf2jQZx346h//OuHf1eDSR+8S/stpwPYOGrsa6/YBmria5NajT9uKuNCW6zrA9u63Pv4W4b/RIPe7jfBC2wkbBBvUdx1ge5ca7+ZyBzBLKNlXASSihzMUKKoA1+++DRviQ2MKWObCMq0HOt5hxzunJmzvcoY5hbB0IzOFfAw5D2jz0Ob1XQfY3gGWNbBMazFkOzGVtPVd1xEf7Cvbu6KDZv01zpOTE1r3Xy/JnNpfX1GwaRIAACAASURBVAUwNd2Qn3EowPUbjQ0am77rAIAZgFnfgwO2d33HO/T4dGs19DSOPX1921fEhzad7R2AGYBZ3/YT9gf2p+86wPau9OgbwCyhcF8FkIgezlCgqAJcv/s2bIgPjSmAGYBZ3514tnd9xzv0+PQeZkNP49jThzYPbV7fdYDtHYAZgFnf9rPvuo74YF/Z3hUdNGOGWVreIgVwtEtqe5+epqOFT3YFntL+drWZIrSv1eX6jcYGjU3fdQDADMCs704827u+4x16fABmZZcC6/Lv274iPrTpbO8AzADM+m6DYH9gf/quA2zv6gFuoRPFEfFxa++4WqJ5QDuKv9qwRXvH6aWbWJLZsXRGCsye7m+T2j0SmRQQiuuI9H+6T9vsrhRJLxHI0qcmHUrR9v4SyFFrrnZJpn7piCf4AL/vfRu2ocR3+Ly1bede3sGAptCHH1IaTxWYXfzq6XqzZaVepDfrjxv8Ev3kZ7ktrY5vXnA+fNB3p3as8b11RWr8Ar3mfSzB9T9N5z60QITt3Vjz3ZZuN89Ck2+/IOqjrH/NPQBm3YDZOhoPpc3LnY5PXj4l6tcZOpTtyNuv0DnRB7z0BgaRq+i/qsZs76YHzL5C5z8nbZki9cW3nQ8bfOMd2UacpvN/2ECzNjsKP7aFl+ncGU/j8xeamcheu/LMlcuN3wx+IIj1a1vfU2kXcd5pvLWsxmzvSg/VmyWZx3u0pXbowOxpdkx7W4p2DipIdrBDqvYLwRmAWcdimhgwi4OwI9qVUMsAq21ahnGFanKYR2a22NLADDPLAknZwKzSiRv9M2+cIXXqDF06pQjArNBApkXjaQKzC/TZz36BLlaQ7M03Fakaillg9tnfDmeWsRborHNnve14gZ478yq9VUGy184rUqIjb6CG8Jeasr2TbtM4b9fEz6OvGYBZW31jv/U0Hn17GR3knadLp16hTyo/M8B5/nw1GDpPl5RoW3VboE7RB28Xamui6ZtCXKtrzPZuqsDsJ/YbCCbzaGDZ516jbyS+DurbQ1yzjZNHC8ye+7Z04/ML9Jxqfoi69OGr9Iy8njowi/Zr297TKdihnvOwgsZs74KBbmaHGpgd7AhApuHZ1h4d1x8E8ABa7W7h2VyAGc9wsoXTQKCUOzkzrbZpf7/rkkwNiKr7za90PEPKn9nF7rZWxNNhn9k9stDJT3v1ZLNssTU+nt3FYUZqYzCjq+Ve8fjRrorOYnPhmA3LdROBeKeuHhy+DkOXXUoPTm/K34vEv3TKXM6us+Hyix3Melv1OT/+jtecjml25tsM/A59cErRpTfsEcCsTatV/do1Zkg05aOZbVYDNACzEgMTF5B5HXlv5hnbuxLpGFKYriY8yKmOkcENgJmnkVdvYmW7rMZzaGPNDAsGaAaQyRln3B6s2p7gOV2HltGY7Z2ESdM4tzPM4sDsbfoJ5c4o8/Mce5/h5tvAFmBm2pBmhvKlj8J2d7r2ju1Y+9jBeU8nC/NL2eTVNGZ713H4u/JtFpj5gEzPKNs5cL6e6QC1OQIzAzRcQGVUT7mTD3cqANNpthPDGjc+HyoZIMThLUiH0gCuWsnoh+NfO+ESkbl2ppP58KcJ23+WAh0SddUHbSY/Tbj2KV/TRFjSOZjVx2lvwjZprpdthv6+PjJ499yWWwP0jmjXlE8VptDQjXPV59zYl7liAzPdxi1u0E1jZn4Jb2/05qZLzvwu0njKoMzm7QJ9Vik69dVfqpZd+ksyT9NP/r472wyddr/Tvuhad9QV1ctBKhj0jFxKImafsb2bts6eJh788WeXaS3037Q1WVSPlvVfXuOctnWYYbkzysIBI9ra9cttOY3Z3vnAaPzX/pJMAcj+8DX6UXWaflQu2fSWa8LWdbF3/pJMMaPso3fJtCNKQzN7X90GV+3N+nU93nffdLiL+rU2fe57uuk0jy3+VTVme7fMGHiVew0w0zCs2bvshI73tpzrk5O4m3bX/7OYYWYAjpwxVMnd6u4CLwrgTarILEARfEVTK9oW0Ms+qe+rwE8qHRWwcsKS6VgUbhSYuemW8CcEZkQaODUQyX22uXLzHAunM3xrAtWRex9asPDK0YOEjov0kmH75zquGrwJT6OxVxdknKs+J6JY9pQNzNgM6lrpNfuq8C/f6MSvpWXql7MOGk8WmP32i82ePvVyTBeM6bzbvc7kHme/DmjhwZ3kwEbunyKA2CXjLjr2BqAp4mUlbO+S4XaNf4j3pTRx0mpBD+vBOuh2g89xbBlMrqFxETubsr99upuZZNVeR/VyTG8mVJUevWQTs7lXAAErasz2bvyALL70kvNl9yt7kX5BL8Hc1+2vD9AUydlosHEtNs5pL5r7zIxaA8gaNwvNFKnIFgiTtHeL+rWJ93SSWpRqY9bQmO3dsuPgZe9XJyd6c3/eu0zsWYYZZmbw4whaQSldOA4AirnHIEgAb5zQxYULj4yHiIMrhz02M6XkEtAmfRFAJNPRIVwDr1zCJNKqT5v0hqDLxt+kx3tUXDYzuSJpNvd1D6sOVuZVhOFmp0k/QznHPwijDt09Sd2n3QOQJvK46nNu7EtdcR2aj0Hnqb7caQUwy1/23TSeLDCrN/mPQ7Em33YGmtzTDB35pjPeVQunI2+Ahlwq4v76zfaua9hjvc/RRAyAUssIdaMx1rxuKt3LapzfznIbNpyjmRmg7I9RmGFWplyW0ZjtHYOl6R71MswKihlgVsEzs4eZnY32o+98pf4owKZsxrjjlT+2yPN3ydpC8UPVJPcw69avZTsv31N2w3GRTVxPY7Z3Sw2CV7hZnUSWXxq3YA+z9JcyZzHDzBFXAhbpIdwNiPJmFaXAiAzCnItw2C8WHvsFR/m8ADN8n0xHh3AXAjMTRgXuAjgUiZ/T4R85LXz0/bsu75TPybwa91h6tF4MHiP+QRgyAnEe5L3yi+ZHxLnqcyLqZU/ZwMzHkNup0pxv58h7r5T65WQ24XbTuAFH4eyryfj9/hfolAqXXpr8RfzG3aFeHnZlya/ck0ueG1A0T2AW24TZ7jPTzLaT2mMPsxXqblDXdBjuQFJqPIs21swOqDb2xx5mnb4Et3S9WEJj7t9MF5RVM8+qZZjmS5jyHMAs3w8hwt6FP7y47ay2e0vX68H3j7v1a+t8y/d08HlbBLL68l9PY7Z3y46Dl71fxfcm8zb511DNAWjulzLnB8wiUMUoL90ttKpnVhloorzlganiksCL77Fh1+Gxc/Qo0yHPq5sdALQ4XH/W2NEuwyUbnpkZxnupidlmxjcFg1rSvb297X0AgG9enFa+sz46edWuNgy5p5ubv0V61SFHTrwy11q07WHmadaUbdfnIkno6MQGpjbyszPsmGFWvuzjGk8GiokZZV/VEEwsw5TLLt980wVn5gua9QcBLDSUA2ycJyCG7riLZZjuTB+v4+4t0WR7NzltWzWxOro6udoCmLl6ROvHmhqXt7N9DWpEPHpQ6C/DrGaYfXrZDn4uvVHdHwA0Ec7s+h1L5H0NjdneTQ2Y/cIXxZLLj79Fv/BFRar+KqY3o8xfovnxt/JBJDF7N2ozRuz/2nl3xphZfslLL/2Z3AamuT/GTNLeOXbK69e2vqdLvO9OHHN/bjmN2d51HP6ufJvaUYmZY/pDAOaLiXqPAm/J5hw3/Tfgp9qvQevCa/ZS7rpIGJLxlyf1dQ1J2sosBsxMgN7XLAWAS6ajCwBiiCTyJ9Mp8mGgjh+XvDfItwvX2nJtH902S2FZXnu/1YNfCj42gKklVJ1WJ32shwxTzgRkfxFmEIbw80+FVhLK8bJVTrubJr+uSM1kOkV5+/Euec3pmH7jlmp4PIOMxqrAL4NxjScJzP64ZWN/ubeZbgs8WKb1mFqnu0x+LBRj26X3qzn3oQAeVeed/eV+XexWJl0iDb0PlBZoUs188jdmZh0AzLqU3XoaT7ONtbad3yulqtll3I6aWRbcn/T8+B4cF7S5q2vM5TI1YGb3KeN6JWGZnHHW+Mv9y7QWbPdwbLF7cr9G3V9hWFa1bfX+ZWZcKz68U/lP097JcYTfr13wnsLOLbBzUls+X05jtndLDoOXvt1+JdMDYLyZf9fj/GaYLa0zHhiMAhEgNpi09ZcQNjDTb9zYAOM4lLKeJjBbb3kpOvAtHfgMEIrtHXR2dQYwc/UoUT+GYneRjvn0AdjeTQ6YmaWW7R8CaMtzifcbYbo2FHZmPnZmKGXN9q70CBrALKFwXwWQiB7OxRQAMNPScv0eisFDOubTyAKYhXANnW63051bD7Z3ucMde3gAZmXrna4faNvm07YNpazZ3rXBozn6jd1ejyH9Q3kHkI752F22d8WwQRUwgFlC4bIF4C2zq6a2cpydlhom0j1c5xJ5XiXMdYDZKvENs0S4rqFRmU+jMpSyBjADMOu748/2ru94hx6fbp2Gnsaxp28odhfpmE9bz/ZujlCsLc9jtyVjSD/szHzszFDKmu1d6dE2gFlC4b4KIBE9nKFAUQW4fg/F4CEd82lkAcwAzPru+LO96zveocenG5mhp3Hs6UPbNp+2bShlzfauDR7N0W/stmQM6R/KO4B0zMfusr0rOmjWK7MeP35Mjx49oocPH9KDBw/o/v37dHR0RPfu3aPDw0O6e/cu3blzh27fvk23bt2imzdv0o0bN+j69et07do1unr1Kk15D7MnT54Q/qHB1OoAG5j/dfIe4b/RYAwdkrGnER2ZsCPzd7d/lfBfTgO2d/RX7xP+hQYAZsWBIexdaO/wDop3sIBNYnuHuufWvbH3ncaQftQ5t85pPf78//m3+C+oAdu70uN0zDBLIEkugIQ3nKHAqBXg+g1Y1sAyrcUYOiRjTyM6VGGHCrCsHCzT2rK9w0DdHahjDzPsYbYJe4z30H0Pc+vB9m4TZTvkOMfedxpD+odc/ptKG4BZWWDI9q70oBzALKFwXwWQiB7OUKCoAly/AcwAzPruhG2q0zLkeAHMAMxyD5q7hAdgBmC2CbvYpW7intWhGvfvNlG2Q46z777OHOMbcvlvKm0AZgBmdHJyYv6nvCSzKLVA4FBgQwpwhwrADMCs707dpjotQ44XwAzAbBOAAMAMwGwTdnETdX1OcXL/bhNlO+Q4++7rzDG+IZf/ptIGYAZgBmC2IdiBaKHAugpwhwrADMCs707dpjotQ44XwAzAbBMDegAzALNN2MVN1PU5xcn9u02U7ZDj7LuvM8f4hlz+m0obgBmAGYDZstTiaJfU9j49Xfa5te9/SvvbinaPugd0tKtILfNA96Bx5wAU4A4VgBmAWd+duk11WoYcL4AZgNkmBvQAZgBmm7CLm6jrc4qT+3ebKNshx9l3X2eO8Q25/DeVNgCziQAzNqx83No7rkHYyckB7ShFrptdhsnLMfURSzI7EpABAzMDyATM6xuYPd3fzgboTFhK1Rs9g/uF9ZPf9zEDs/+296wp47N777hf+jzYqsteqWfpN49dKJZ8Dpv+Bx89eOvKaaPlM1cuu37ffkFofJrOfSgGnm1+H71Lm+q05Ir3k5dPmbyfe3nHzcsbZ4Qmihz/Nr/LV/CFTO8rof/j5z9jtHzp53ddbf71jzsaO/6O32foP/xuA+HY3o150Pz0fWvvtt//mvOlT3bnPO7+Ae999DXaP9u0g8b/4kXnWd0yzHFQl8pzyt6xO2v83LdT9k6Rbytz2Z1NhZOyd+zOmlx6I/yYyeHztv45tvDyFbcOFvhK5Njec36Hc73bXCabqjM54uX65dcdHfaqfqn3fq7ubNd8m6X1iPtdpnNnvDbl/AWnDclR9psMY9W61fYcgJkLzP7om7Yv8/lv/nLw9dCUH7tb27ZFvyO+usn2Lhzp5nVpNv0/3qMttUMH1b5kx3t60LlDe3tbAGa5NB8wMPOzOFpg9nSftgX4s/Bsl5aYXOdLMclrNjDjBGYX6eeUorN7F+k3t/RRADMDywQkM9dbdPtEQ7OW54w/vpLZdBwv0HNKD/4umA6S06EyQExAMnP9Ar320bt0qc1P+48amJ2nS0qDsPP0wSkPiF3Wfqfog7erQePbr9C5+rrNz96PGWYMt96if6kUvfTzb9F/+Hv6KIGZ9hMg7Hd/ml7iawPLhJ+5/nG6V4E4tndjG0jb9F6kXaVo+/2LBoA5g+qPX6fts6/T0wo62IH3WToy1xaYNQCNQZo4AphVg70We/fhq/TMmVfprcp+2YFkZe8+0s8JW6jvlddTtXfavp16hT65bO2XHSyeocPq2gx49Y8Ep87QpcBWApg1dqjMu832bpPgYfW4F7WzbW1wys/W06Z/I4B39V7Py6/F3hmbluj7fWSBmfODgaff6uVe9Z2kDen1vFy9AzBjYPYl+hml6PPf/BJ9/UV9lMBsgd+L/4T+qIJkv/OzitTPfqmGbWzvSg/Wa2B2sKNo5yCcPabBGWaY2WJwZy5t0361tjLlThre1DOdtml/v+uSTL2EUod/ZDrKtjJo6GOXVtrrJv4qdWbZJVccpSQk4iWZMjz3eR+Q+dfkxK1/YVgu/PTzMk/2l4ttFjZR+1Npiz5nysDNayJYcstLLmH10yjzTqs/l0xIeQ+uJ+MEZjxj7J0AmJnZY1uv03+rAJiFZAKgGffwOdZhXp2mLp1G20GSwMwMGMUA8pIYNLb5sbbj7VBxh24nBGYGkMkBo4BkbX5VhxDAjIEZH3dDYGYAWQPB/u52A9DMjLS/99P0P+qZao2f1pbtXTNQFdBoNDNcLABzgJmfdg3Q1LO0/7HOH4AZ25zux9DeBc9KKGbOGZ5pe+oBtFEDsxZ75w9knR8I9HPWRl56I2IrMcMsMsMu77vN9m7cbW287tg8reYXvMse7Jmff5u9i/lZt2kCsy72brV6B2DGwIyPvxwBZl387D1mtpkAaGzvSo+cLTDTs8u29ui4ml0ml1sCmFVFYMCLB0q0V8q9AkwNxKlglZj9lC5cBjQcH183EMdAujqsyl+sPXRnVvHzDThy/Yl8CLXoOhq/asJf7nkt4xJLMvVMPQns2qCYf29SdFs+TnkZfRdpu+pzyYT04sEGhkHROI8R8OXMKGtmlf3cAUM2fYw8hxlmzrT6puMY6TTJGWWmw2l/rTSdqDa/qnM67k58Mxj0l4rY5UcamoWdqjY/rQeAGYMyPkaA2e1fpXv/SP+ooqGZ5+/NKLMwTdG//Nc2PLZ3kwdmf3CWlHJnmHHe9fJ0C9IaWIg9zPwfDSL2zh9UezbutfO6TmpoFn92qvbOyZdZct78YGBmnD1/vgZnvq0c93vYvD/58tEBmC3xbvM775SRDzkHfx22o01+VvNr+jX+ez/X67jNsjrF/Kwb1y/lzabVzzVlxABqbMfV6hb/SODbOq0HgBnDMD6uA8x4JlozO43rY+nBswFmenZZahYZgFlVBAbKNMCqLphWdwZe1d2dl2RaSCP4VwiUJAiKQjsNchhgheERSf8FwCwKpOTzkfBlXhc+vyQwM2lvysKFd3XJaJppZt01EEz6eedST+m1SNtVn5NxbOCcDcw4QRnDrzj4ur1jZyraPD5LZ7cUAZit2iGMdZreJTtIZJ1P0zNnFPGvjm1+U+9Q8Z49SixX4g5kmx+AGYMyPnpArJ45xtBMkXJmlAl3M6v7M/TS35sbMGsfdLvLNe2gH8DMt4txe9cMtOP+tc1zZt7asPn9H++xbQAZ+QHBmVEbfzYfaCoBrzYRZvu7y7NFU7NL/Xeb+3fjrXOReuUAvni9svlN+zXvsf/ez/U6bs+sTm1+Vi93efoc7F26bgGYMQzrclwBmF0Te1OL5ZgaRrK9Kz2UVnZj/2bvMjm7TJ8DmIkiMPDEDhIdCBNzj4EUCZFEsOFpCKCCGVgyfHleBybDkOd8g50ZxVCudUaYyB9XTHtsAXIyrwufXxaYScAXy5vNo8lTPQuP8504yvTKWxZpu+pzMo4NnHM5ThGYuXmy+5YBmK3aIVzcabLLkBpg5nZKxeyzSc8ws/tf8MbXdk8f3tOszc/++gpgxqCMjzFgZvc341lj9sMAYt8yAdXmOMPs6KIiJfYzC6GE3S9J7mkGYObbxXZ7Z8CYA8Vc+2YHkGJPs8nPuLhC5oeA+gcCO6BkO5gaRIZ1cxOQakhxtgOzZd9t7t8BmLmzm9y+if/uz/G6zd61+bFWrv3T+o67zpUBtZhh5kO0FYCZ2OTffgCg2fif7V3pobQ6OdghtXMgvozp7mMGYBYrAhc2NXcIdwOJpjTDzMtLk+l6JhfDN+MlQVJMC+f55YFZvRQ2EfZSsEynJQrGUktuxey6VZ/z8t/3JRsYFy7xzK2xHOMzzJw8BUs0sSRzuU5jh06Tt0TJCT/iN8UOlQFk9aDR7XS1+bEWAGYMyvgYArNwn7LwnlpHb4km27txD9TTg+rFA+r3iZz9zSwsADDjgR8f0/YuhGXVl+QcgBY+z+/4eI/pWRUuLNN2z/44wO+bcxT2cdzvYQnQlvfdZt3HW+fcNjTMR7pOpiCtDsPpm/hLrWd5HdqrRqM2v8peyv0cJ/2DqK6Pq9dJALO8wOzPP/on9Hn1LH39Ixsu27vS42iV2uyfZ5oBmMWKIDWrSbpbeFbPRONZVp1mPMlwbPytM8x4Q35BrNxlijY8VS/RrACVSEvrDLOFSxvD9BoAVYdv/WstIpK66Y3cEDhVYW77e59Vea3jDh5MOHjlpZd9mjCq8JLarvpcIhk9ObOBceBSvVH+VIAZfxVTfEUTm/4v2Wlc1GmyvzDKjwI0Ha64X9j55c7IWI6Rzrq3h8+nZlmSIjPTos2vWmZSgx5nlhTDozkeIzDMg2B/Zz4C0Cy7bDS0M9HkFzbZ3o17oB4bVFu32Myyo4vunmUxqAZgxqCMjzF7Z92UA8aq+/0fBMwA0p1tO0l7V+3TGFt67uY3Yiux6X/HTf9Xf7fZ3rllMZb2ldMZrzs2T6v5NX0Tft/nfozZO9Yk9HvtvDt7NvYjwrjrnK57q9WttucAzNYEZhqQiWWYG5thtqO2aO/YnVXGsAxLMlWDL8xMIt6zR5FigJJy108yJDN7quzSkb7uBHIspOEobFAeGApmNllwww2lcuLR4W3T/pH8aqc7Y6wdmJkUeF/hVNTEEabXBWaLnne1agNrOiT+M5BNNXuZGXe/PPgrpVJMDsA/OuXFy031TW3aummXUHLhc378PV5zPRknMLMgjPPAx7N779D/On6dznKZK0XGrQaBLc9h038PpFnYxdry0YCxalDouPGvs21+o/8FMj57gjd5rfcoq+ofu+sOZJuf9m9gzxzhmMyzhV1ct/jI8Mtu+t+0w+zO8My/n3Vl93ECM7uUkvPAR7OfkdkIvNGD/dTFi0S+X2y5JpH33vNgaW7HFntnoFhE4/MXjHb1/mXVe+//eDDeAWSLvTM/AkQ0MRv9M/BIDz7H+R6WmFlW5t1mOzDOutdS7xIzGG1b2/acrZMAZmzXW+yd+dJv+G4bu+bbwsiPCOOsc7p+tNWfVf1svQMwY2BmN+xn+8THz39Tb+Df5meXcPL9+gNGPLtMa8vupYfQ9iuZka9jSmi26Py73/1u6XSGm96bVXSKusKVZRPYVwEsmy7cDwVyKMD1e5zArNwMOHSouENV7jjeDpXt/JRIP4MdHCU8y3fO9g4DdW/QD2BWHBiWsBdjDxPvofce/lXea7Z3Y68nudOP/l25fh1rm7vMphAegBkDszJHtnc5xsZtYYwEmOkZPnLGT5UlMyPInSnVltll/PoqgGXShHuhQC4FuH4DmLnwjRt9HMt1rKbQAcqdB4CyfHAspiXbOwzUvYE5gBmAmfP1wXI/CkibiffQew8BzHrZLB79unL9OtZWvuc4xwyzPmBh3b/LNUBOhANglhKmml6f8F7T2VviJ5aQ6YIvNWtuzUT3+HgJfUqE2aMkmaNiAwNgBmDGHZ2+juhEhYPSGOSBWz6IxvYOA3V3oI49zDCA3IQ9xnvovoe59WB7t4myHXKcffVx5hzPkMt/U2nrAxrNOQ62d5mHyUFwIwFm+iOGek21nE1WAZAue1MF2V7s0FcBLE4J7oAC+RXg+g1gBmDWd+duU52WIccLOJYPjsW0ZHuXe2A69vAAzADMNmEXx/7eDD39bO82UbZDjrPvvs4c4xty+W8qbXOGWX3kne1d/pGyG6J6/PgxPXr0iB4+fEgPHjyg+/fv09HREd27d48ODw/p7t27dOfOHbp9+zbdunWLbt68STdu3KDr16/TtWvX6OrVq9THHmY62Raaic0InY3t3Yyte8UF8OTJE8I/NJhaHeD6vakGZKjxzrGD03eehz7YQPrKzn7YhL5s7/73//0bhP9GAwAzALNNtMV/+4P3Cf/lNGB7B41djfvu68wxvh9+713Cv6vBJmzsnOJke1d6nD6aGWbrArBln+cCWPY53A8FxqAA1+85GdUueZ1jB6fvPG8CmCDO6UGwZcqU7R1gWQPLtBYAZgBmXdrF3PcA5LggJ7cebO9yhzv28Pru68wxPsAyF5ZpPXLbT4TnbmvC9q702BvALKFwXwWQiB7OUKCoAly/YXhdwzvHDk7feV4GdODeeYOuXOXP9g7ADMCsb3uHNtZtY7UeYwcvQ08/27uhp7Pv9PX97s8xPgAzALO+2zy2d0UHzUQ0HmBmvogplmPWG+XLfc3yydVXAeRLMUKCAt0V4Prdt2Ebenxz7OD0nedcEAThAKZ1rQNs7wDMXGCmW4y+3/+5xTf0Nm8T6esblMwtPrZ3c8v3ovzOzfZsIr8AZgBmfbcpbO+6j4BXu3MkwOwp7W8rUoU2+I9J11cBxOKGGxQorQDX774N29Dj20QHY25xdoUcuA9ALFcdYHsHYAZg1re9HXqbt4n0LQIb8F9vySbbO+jo6tj3uz/H+ADMAMz6blPY3hUfN5+cnNC6/+U3/ddfxCwzkywlcJECONolVfBDBam8EFnguAxvNB9YWOaBdOTwGaACdLLOVQAAIABJREFUXL/7NmxDj2+OHZy+85wLgiAcALWudYDtHYAZgFnf9m7obd4m0geQ44Kc3Hqwvcsd7tjD6/vdn2N8AGYAZn23KWzvSg+1FUfEx629YwPQjve2iN2U2qGDFrBWHphp4LNN+09Ly9GEz3lvXDKcDRiYGUAmYF7fwOzp/nbWGYQmPKVou89Kk6GK9BUE1+++DVuu+D55+ZSwT4ouvVHtk/LGGced86nUGTps87tsn59jByeV57eunHa0fO7bYoPsb78g/E7TuQ8bv9bnPnqXukKOId739P1nRb4V7f5BBa3+4Kzj3tS7s3T0V+8Tef7b739t1DqULJsSGnN5jBWY/c9fcd/FL/8nF3yx/0u/8lXnK6DsbvP/D+i/el8Jxab/3ezWIpvG/s9cuRwscc3V5m0inGQ7W7WX7H/u5Z1gY+s2v7GDl5zp/4v33DblF2+5MI39z733tejeb3/0JbtVjfRne5cznX2GxXnmfCyjifvss/Tbf9bomerrzNGdbRZrLPt3rt8L9NpHjZ289OGr9Ey9JZIi+ZzWcczA7Pten/fLv+Xmh/1funI5ms975+276PtvwnYPNU5uF7je1WO3BW1K23McVunxc7Mk83iPtmowdkA7Owf1zDMDz8S1PyOtPDAjIg2bepxlVqQABgzM/Io2XmCmZyNqUHZklvECmPkla6+5fg/VqLam6+1X6NypV+gTx8CeocPq2n/28HlF6vnzQYde3+f7zbHjFM2z7hSdeZXeqjpKtgNVdZwMLBOQzFxXfm3PVWGVhC1Fw/74ddo++zo91QDsr94nC3YqIFa5yfiPLipSFy8S/dVF2lXP0v7HFVzT4cjryLMynFmdF9KY7d0ogdm91+ilH3+N/mcFuywEY/j1Nn1ZKXrpV96m//jj+iiB2dv0ZfHcf/1nitQ/e9sBagBm1UCwzW61+X10gZ5Tip65coHOndHHCQGz1nb2PF1Sis69fJ4+OKWPEpi1+dkfpvqEL4OO689ep3Nbr9Nf/MBCHQt7ztIfmeuL9Ita4/cu0m9v6WMEmN06S2rrLP2i58/2btB5r/IcpHENTYx+Qk8/7GhfR8KguZwvsmmi7/eahkDnL1Q/BLC9q+yc3xccMzA7eJVeOvMqff97FpJZOPYC3TPXF2w7e+UCfeeMIh+IGUj4Wy+QOvMCfTni749JZnu9apvS+tyV+sfq+Gg3n2sNzA52FO0cJJZnapi2tUfHiVlm5YGZhSDcCLjHMks1OQ4pNc9asn7NjLeUOzkfKtim/f2uSzJ5Rp3Mt85ntZeboftN/DaN0k9TbqmL9ds9kuG5z/uAzL/mZZ2sy7Lhp5/30714VlgqbS4cs+G6brI0I+dOeSlqVqT6aZTaErnlvMRzkST05cTlOAnDrY2pOkUfvB1+jevTJf3QoRK/JMrOo+5gKQvJDDwTHapLZtAoAFriOdZ2MgCoDXxJP3MuwZoH0ADM0rPtpI6+TtJvgcZs70YJzLxZYf9bAzR1mv7jPTnL7KsRYCb9f4MMaBMATWsBYLbY3rHdqo/CFtZuH12eHjDzf4CKtqU7EWDG7XDazwcZuK5mQWlYpNxZUX/7g68lgJl1/8VboT/bu0no2lkTDRh97ZrZZVqL5n1NvPey7zKn86hNsxo5/T3546jRx9o9OctszDPMnLRrgKZO03cO5CyzywlgZt2//Ftx/0mMs/z2IMf10m1K1bZ4z7G9Kz1+tsBsARA7OdghtekZZqWV8MIPCsDAFA+U6GdS7hXcaoBNBavEskcvSnHJgIbj4+sGxhhIV4dV+TeEhyzE859vIJnrryfwuR9VWHQdjV814S/3vJZxiSWZ/mxDUwZN3FZIq0mjv5A3emrLp7n/iHaNvou0XfW5aCJ6c+T6PQlDbpZaxmeY+TPIZH5jfuhQJTqSsqMkz02nyf7qKDtNtY7BveNekunAPrPMUoKwZk+xZnaZdTPXSt/7Ndo/qwhLMhutHE19KJZJY7Z3kwBm/+kfkFI8w4yh2CJgxjPR5Aw0ALPaTvmD44jdqu+N+s0AmEXb2TQU+/Ry2m8SICc1Q2oddz1jTPEMM4Y9IRDT+pnZVF+6SDGgxvZuEjp31aQCa+e27LI4o4HRh3UEMKttWGd7584oc+CZAGZyZq0DnarZWqN00zPGFM8wY2gWB2JmNtr5C/TD78X95bgD5/yDyhX6dOk2pXrWe47tXekBtAFmenYZ713mL7c8cZZqxmeglZ9h9pSe9rh/mRY9KIBq9pFgUrZsWt0ZWFXF2HlJpoU0Mq4AKEloFIV2GuQwRArDI5L+C4BZFEjJ5yPhy7wufH5JYGbSnoKH/MrYNDUAjN0TR6mnvGWRtqs+J+PYwDnX7/Eb73Sn/NPLdmmIv0be5jnul+xQ+B2MWV2Hg0EzTb/ex+I0PXMm3MviUmLWRSsg8YHJYK/bwJeeQSb2N6vyYKGZIiWWdU5Di47wa+myzKcx27vxA7MUGEu4G7hWDSC95ZiYYZb4cSBht2zbENrCdvd3o1sBjK/dTbWzKXc9sEn7TQLkrAPGos/GwVgMiP2tgUMM1sLn2N6NX+cwbzZPEXcD1sQMM6ORIrn/Gfp3MZsXsWnmR4Gq3aiXY75LITB7l3RfcHrALA6+okDMzERjsBZ/bnz2XkCtHDPJomGk2oeUO6cp9Gd7V3oorU5ODmin3rvMBWJm77KWpZgM18oDMzuLx4rCEKisNNECMPDEGhEHxMTcYyBFQqTW5IcAqhWYxeJyvowZhmeBWQOdWmeEifyxLm5ZRMKXeV34/LLATAK+SNxGW+vulFOb5jK98r5F2q76nIxjA+dcjmM35GaWmNjPTObHbBK5pB86VGGHysAxZwmmf098hlnquSlAIgO/EuDL7G3m+LkAze59JvY0WxoklQJUwwo3p8Zs78YOzMw+ZN6ySpunBDATyzndvc/szDTd9MDmufYsZbe0Tmm/yKCz+lFFtkljPU+3s+Hgpclj2m/8IKeZtZQrL2bz/uj+Wz4cstcNCPL9369/8M+Vtk2F012T9+lvg5looS6wda6ta7dp9l4DyZTdozYEZqHdG+VsMm8mnNm8X+xn1uTJB2L2uvk4gO9vZ6Y1NpGhD46rtSnVvtPeuI77d6WH0iq13FLDsuSsM28vs/LArJHBgKN6ZoM3g6u5be2z9gKwAE/OALMRCncDibz0peBKkNoQArUCs1hczgyyMLzlZ5h5eXHSHAlf5jWaPieA5ZZk6kc5TD66wekbltv0PwrGRDxO+GJ23arPOeH1f8H1e8yGPG1wdWMUn0Fm85v2Q4fK7VClB4jivsgSpbbnxg7M2kCO3eDfnV0WArS2mVPDglabKqvcGrO9GzMwS8MyDb8WA7PY3me65YHNa2xZm91q80vNptXajrmN1Wlvb2fTUAwzzLpDtTQY0mH44Md+DIBtmnOsgBu7bQp05Yh3OU3eJzvrTswwC3TDkkzf1rfbtMouyv3Ngr6eBWZyO44GLvEyxnEd07BM58MHYvZjAPy+OUcB3MbeBuRO/6ptSuo51r30SFpFN/vXyzBb9izjmWV87BOYSUFKwrP2AogAIpMw6W7hWT3DyYAdRared0zmxD+X4Vi/VmDGHwMQBM/cX8dlw1P1Es1qRlftL2ds2fjcGWf2+TovfnKd2WyVpwRmHeCVm94ggohDlabt1N5ni9LsB+mVlwaObXuY1dqt+pwff7/XXL9zG8J+wrOddOX9yiDjtp8gju9r1ubndyjme207Qqp1ZpnuULl7XPDAse25TUGY9eO1oKttSWX0y5n+Plxmg3oXqq2ftqmAtjIas70bJzCzMExFZ5a17GGmPw4glmFihlkDxkK73mbv2vw4THuPXJrEcch2aVzni9vZNijW5pcDqEwjDAvDVHRmGQM3H5ixOx9Df7Z349RoVU08Hfwlmtj0X/w40mLTNCDzl2FWM8y4v1cDsgCgjQuOuXDPwjDdd+UvZbr+MWDm59cHatZ/XHa/5Oy3VduU9ufY3pUeRasdtUV7x+5STDPrrJ7FxRsoRu6rZpr1CswYPFXps4zIQgv3y43rSRcUgJlJxFqIDfJT7jp6J627dKSva9DSlj4LewT/CmdgBTObWIMqjU48Orxt2j/ap+26XN0ZYy4gCwEaz9hiXcyxjiNMLznAzIhhZnzFn3e1SoM5VzMGplInXmrqxKMWf3nThOyUl1z626atm3YJJYO01Hq5+djEFeszSkNuNnwU7yLX6efPV7/m2xlk7qfuuRFo87siOhQ8EJrpUe5hwfrqo+5ImV8cG/2dQWLbc9USpdHCIQO+mnzzO6QuXqy+8miXXsY29DczpoSOsXtGq0vOJaWFNOayGiUwk/uQiTqkDAyzm/lz/vj40q/ozf0r0FY/439ZE5v+M9S61Ga32vyqHwxYdz5KmzjKNlbvOdPaztp2lPPLR9vmtvnZdnicIIcBVcajgTqRNsVsVh+fSXbuva+Rq58Hin4w8iWZ62hS7VvG9bFZtmrLrH7fZ7UHbaQP22rTKpgm2o1zH4ownP5f+HX0EDL5UGmg12aT/8i7aDbzj88ke+nKZXLzC2DW2t6t2qa0PnelXoJeeixtv5LpLbHkmWNdj+WBmQUybASVS0iKaMRxFQkcgUKBDSvA9bvVuEU3amTwNM0jOlSiY1SoUwkwNJXZYOPJB9u7UQIzsQ9Z7vT/3u/9Hn4kKGTnuC1BGxv2FVzgkxFARTfTn1/4bO+gs1v2/E7iWK6f5wKkgcIxb8+y0mlGGxC2ATk1YXtXelg9EmCmZ/i4M6KKC1PR9dLxIHwosAkF2MDkNFpTCAsdqXIdKdYWwGw8oGkqZcX2LjdwGnt4AGbl7d0U2sXceQDIcUFObj3Y3uUOd+zhcR8Ex3J2rzR8GmP4ue0nwnMBHNu70mPpkQAzu0SwnlhWL52Ty+bySlW2ALwlfvXUVzsdtOuSxLw5HlJoJfQpEeaQNFsuLVy/YXhdw4uOVLmOFGs7FQiDfIwH/LG9Gzvgyp1+ALPy9g5trNvGaj3GDl6Gnn62d0NPZ9/p4z4IjuXs3hiBVuk0ow0I24CcmrC9W24UvPzdIwFm7gwzvd+WgUrBPlnLC5B6oq8CSMUPdyhQUgGu3zmN1hTCQkeqXEeKtQVoGg9omkpZsb3LDZzGHh6AWXl7N4V2MXce+gYlc4uP7d3c8r0ov9wHwbGc3SsNn8YYfm77ifBcAMf2ruSYWYetHj9+TI8ePaKHDx/SgwcP6P79+3R0dET37t2jw8NDunv3Lt25c4du375Nt27dops3b9KNGzfo+vXrdO3aNbp69SqV38NMADMzu4yXZwr3zEpxATx58oTwDw2mVge4fsPwuoYXepTX48of/zrh39UAHfhyHXitLdu7X/2TbxL+Gw10twl1r2zdQ5tSvk2Bxq7GbO/e+9NvEv4bDWDryto6rS/eRfddNHp8/Sv0Kf6LacD2rvQ4fSQzzKqvPD7VH57cpnrTfwee5SVmXAB5Q0VoUGAYCnD9RuMWadxm+LGDPusBYJkLy7Qe6MiX7cizvQMsa2CZ1kL/oe6VrXt92lbEhfZc1wG2d4BlDSzTWsDWlbV1AGYJ+wNYVgyWaRDJ9q706HokwIw0KaNts9cXzy7z4FlmpfoqgMzJRnBQoJMCXL/RwU40cIBmxX4pBDADMOt74ML2DsAMwKzvuoc2Fm1s33WA7R2AGYAZ7N0A7A+AGYDZyckJ6f/ySzI7MYCsN3GDkzVQBAYFBqIA1+++O3KIbwCN94ZhIIAZgFnfnXi2dwBmLjDDHmaYcYE2eXptMts7ADMAs77bWtiTiD0BMAMwAzAbCP1AMqDAkgpwhwqNW6Rx2zBQmnqZAJgBmPXdiWd7B2AGYNZ33Zu6PUf+hteHYHsHYAZgBns3gPcTwAzADMBsSUpR8Kue7SnRe8Ap2j1qv0v66i+R1nvFSY9Jn1udTGdje5+eTjiv3KFCZ3cAjenMAB2AGYBZ3514tncAZgBmfdc9tLFoY/uuA2zvAMwAzGDvBmB/AMymAczYsPJxa+/YLLM82FH1Rmpqa4+Oq+WXDMnkEUsyO5KVAQMzA8gEJOobmDkfc+goZ9ttJjylaHt/Ceyly0c1e+S1hT92P37f++7IbTa+HfrglLBrek/E588X26trs3ndVCdhscZTB2bn37R17NRXfyn4GmjKr+9O7djje+281fiZK5erjZwv07kz3rt9/kK9yTPbu+kBsyv005/18v1Pf7b6EmibnwVnWJLZZUlme93S79JbV043/WV1ms592IQ79Xbg8Hlb/869vOO0pZ+8fMpo4rtPXY8S+VtWY7Z3kwRm33lRvGuKnv3lK82XQG9/gZ41e13bOrn1HQCzldr6b7/gaNy0s76te4Fe+6ixdTquEvV/82G29Wvb/Kp+OIBZB2D2JfrgR7y+zOffdJ775Kdsm2Lt2yn64JL9+ijbu9Lj8mbT/+M92lI7dGDA2DHt7R0YcKbBmIFnO821hGX6HMCsYzENGJj5ORgvMDuiXQPKjsysuqWBmYCGviZTumYDs/mGqE+wYxu2S2/0Gefc4lqs8aSB2W+/SOqzL9KZzyoKgFmL30qdWq+jOpswdGf+zAv03BlFTUfeQo3nvu123lkTtndTBWY//oE7c8zm0wKzuB+AGdeNxcf2umVg2ZlX6a3E+zjpNvaNM6ROnaFLpxQ1YOw8XVL6+rz5gapxn1tbmCm/K2jM9m56wOxnaUudptdvVyDMADK+1n4CoBmwxn72/sXverz9mNdzF+g5Cf0/fJWeqa8v0HPC1pkfrsQPU1qnadq7tn5tm19lAwDMHPClv2wZ/ltgdulnYn5fIQPLfuQn6ZPIs2zvSo/Ra2CmodjOgd3E3wdiJwc7pADMiGct2cLZJp68lHJvvuypqek27e/vkuoEZPTSQB2+hT82Pj3zSSwZ1OE5k6ekn45PzpSyfrtHMjz3eR+Q+dfkxL18+Onn/XQvnhWWSpsLx2y4rlv6dXLLkJej6jD8cpC6LdY1HaP88qul6s2SWV8TWZZrPCcSwwZmmo1bqqPaoWGb2fLI/OW/WOPpArNfolc+q+jMb9ujC8za/H69ngk1r475KoMThhf2CGDWBsXa/Cww000C6tyiesh1LnafN7iMQLP8NjbVvvXtzrbeHkMwlnLvO51jjm81jbl/NzlgZgDZi3TxT3nmmABoBpBJvyv0+ucUyVlmsHUxG+a5GUAmZ46lbVzsx4Jp2jt+D2O2pM2vuj8CeUJgFAdF87mvDZi9SZdUM6PM14TtnRjiFjm1wEzPLmtZdtkK0+Yyw+zpPm07EKoqj5R7BZgaYFPBqs7ATEKpBqAwVDGApw6r8mdPw1S2BTTj5xvYYwFRA2J8CLXoOhq/gHjLPU8WRor0t9Z2f+mkKYMmb/ZZm+dG/9YQrWcwA3CRbqG/n+90rDwTjqnnEe2a8qzCFFq4ZbXqc25K2MBMs3GLNWrazTZsnHelDfDbqXvhvlrdWKzxVIHZzldPk3rzAl354xCYtflpPdCR9zrtEfCgNTIddPOLdhyYNe+2uyyO3ac6w4zzp9Rp+uk/4Nlm/pJM6Qdg1v2ds3VNalwvuaxmXzwjlwPPYsbFFTJLLs2WBtbmA5jl7zOsqjHX1ckBsz/9Jl38P/TYSIMxC8R4SeY7v3ya1Oe+QO/UMM3111p0f+e7tUdTDc9ueaChmd/OSl00SJOzvK3fav3G/O9O3nS09Wvb/Kp8AZhFZpT5gNBfkikA2aWfpHPqFJ2TSzbFck22d+4oN/+VAWYaiPHeZfXsMj2rrFoLnpx5Vu1rNoslmQbKRDbOb3VvgJQpugDIpArUQhPBTEKgJKFRFNppsMIQKQyPSPoT+aDHuY4CKfl8JHyZ14XPLwnMTNqbsnDhHWtq05QDmMlycHVbkG9OSuwoy0/6LyrLVZ+TcRDV73beRmXojZ6bPrvHyhk6xKyyYtPoYxpPEpj9/hfolHqRzv+x3tDfA2ZtfuZ+ALNOAw/nl++2jjzvs9L8Ss59mekBM4Zj9rj7f+q9tF6kt/7Eddf5jvnpZqGT9gmAOcdn7X5lVd0ye/0IOGvqqCK5NHiSbezbr9A5xW0ngFmRMl5DY7Z3UwRmOk8WmikHkIXAzN7HQA3ATMKuxee8T6gSSzCNvZf7m3k/Dmj/Iu/CwProsX4t5zvqB2DWAZi5AM3uV3aGDrV2P3OGzASHas+yTw1AU8TLN9neecPc7Jfq5OSAduq9y+JLMs0eZi0z0GYBzLT0BmbY5XMOiIm5x8CGhEitRRmCGAOFJLmR4cvzOlwZhjznG+xMJQ7SAWTkATSRP66Y9tgC5GReFz6/LDCT6YvlzRRWhj3MYmFL3SL+Mt8sdeyYum9RWa76nJcGLkc28vM82r1WsKeZCxLz1oVQ4+kBM15uyV+/lMCszY/vBzBbDF4sIGtARDswu/SR/fWb72d7N3Vg9qt/8rP040pRfN+y0A+b/i8eOIZ1U9QtM3hswOylyIyMvPa0pK3uGrYFZE27CWCWv4zX05jt3fSAmd2njJdZGkhW7WkWAjPMMAttVxd7J+wbz+qu9zBzn3d+PKh+VMn/LnS1S33eF/Zrm3xH/ADMlgZmn35dL8OsoJgBZhU8M1ra2WjnfupLJly2d94wN/ulWrQ/mZ1x1g7VZgPMavklNKkdNWYyG84bCGUg0ZRmmHl5kdmulp8yfDNeEuzEtHCeXx6YWXi5S0fJsC3McsCmF2dwKdNsPCNAzJmZF/EPwghisQ76vs5LfHW9quDkqs95yWAD0xj5PhubgcRlfsHFssyidSCi8fSA2QU6I77Mxe+WOX72NJ1K+n2BdjDDrOMMJ9uJd7RlXf1fwHXHvVoqx0vn+LnJA7M/+AJ9xlmWKWaaRfwAzNwBYKcBpqxb8twMGEOQW9S+bmTmhR0Q8jvlHE+9Qp/UaUqBtIG0/3U6h5ie9TTmMpkaMGuFYtjDrGNb2m7zwn3JQptW28nA/s1jhtmnkX5tbedjfgBmywOzahmm+RKmPN8kMIvuT3a8R3viAwDHe1ukMMNMIIcIKDG+0t3CsxrY8Cyret8xEVxwKsOxnq0zzHhDfkGs3GWKNjzz4YFqyyzXX87YsvG5M87s83VeOqSXHHC06PkKmHXShiOvwtzeJiXyzb78kYF0mps76zMnzdp1kW7W34k+CKMO3Tvx6ocGcW17mNXarPqcGz13qGojP+jOY54O7eHzLhwzn2p3Ovh54pmTpn5eu2g8PWDWzBSzeZMzzLr51R1QLH3r2Ol3O/GvnRfL4j56l8xyEgHS2N5NDZi99U/dfcne+qeK1Ge/QLt/8k1q82MdAMzaB4/6vWyvW249vOQv0ZzFEqUUGEu5o531283F1ykt4+5s76YGzN7zoZj5CABv7O/OPgvuxR5m3dpWf9asgWLVMnN9LpZhzmWGWVu/ts2vfq8BzBYCs8PPiz3Lvv4VOvy8IlV/FdOdUeYv0WR7545y81+pHbVFe8f+Ukw9o8wuPTQJaYFlegbaLGaYmZk9QhOmJCl3XVYMyYyW1WyoGny0FWYIYtqBmQ7LghSuOO7XOHV427R/pD9cwHlwZ4y5gCwEaA084ueV+OJnmF4XmBkxzBLJOn06HVILoVVXyGU0Uc1eZlZRT4cqv53CDGAX50uGKXVjf1GWQRjCzz8VeZYws70s/XrFy2J14DKdnr5e3FwOtUGfATD7VH+eva7/ihRgWf79JjpoDGCmyP2CJpZkLg8MY6BCtE0Clumw+b1nUDSZ4wcv1nkzeaxgmclfm1+1xxmA2WJgZiFYum7Z2YyNPy8D5jo9/TbWhzbx2VHhRwEAzrrXjeU0Zns3OWAm9y+r+nJyj7L3KoBm83+aXr/NX9O0R34ncWy3e/X+ZZXG/teouX7pj8zwLG7WtHudHtH739avbfPjcRWA2UJgZiFY0442sKza26zat4zrHu9fpr+YyW7eMDf7pf1KZrV5f73h/5LXswBm2aVHgMNXIALEhp/oTilkAzPJxo0bKRzzA7EMmk4fmPmzyhZfc2cTx/aO/Kr6sL2bDCiLbOq/St4AzMrUN1lP0caOaGCcoX0bQnmzvZsiMFsnT/K9xHkZ2zeE+j+4NACYLQZma2jE9q7T4HeNmwDMEuL1VQCJ6OE8CAUAzAbX8EykQztnXQHMQoCGznuZzjvryu35KlBpys/oZpY1wrFMHZyzrUfeNwML2d6tA5em+CxsXBkbJ3XFOx9559eAQXoGFf7bNWB7VxobAJglFC5bAN7SOblMTCnqtHwwke5pOJfQZ5Uw1wFmq8TXX+lx/UbjFmncAOWKzkwDMAMwkx3sPs7Z3k0Zfq2SN93i9KH/nONAG4s2tu86wPZuitBrnTzN2Q71lfe+6/oo4gP0Kgr92N6VHkEDmCUU7qsAEtHDGQoUVYDr9ygaGwCsogCr7zoAYAZg1lfnneNhe7cKVJryM7qRYY1wLDP7om/7ivgA6NjerQOXpvgsbFwZGyd1hf2J2B8As2kAs8ePH9OjR4/o4cOH9ODBA7p//z4dHR3RvXv36PDwkO7evUt37tyh27dv061bt+jmzZt048YNun79Ol27do2uXr066U3/nzx5QviHBlOrA9yhQuPmNm6y4cd5mc7VD+/9GuHf1eCvT94j/JfTgO3dD7/3LuG/0QB7mJWxcbLtQBvrtrFaj7/5y/fxX1ADtneoe27dk+8lzsvYPtQ5t84ZPX7wPn2K/2IasL0rPU7HDLPEPB4ugIQ3nKHAqBXg+o3GzW3c0Ikq04mSugKWubBM6wFYVg6WaW3Z3gGWNbBMawFgVt7eoY1121itB4BZWWDI9g51z617sh+C8zK2D3XOrXNGD8CyYrBMg0i2d6UH5QBmCYX7KoBE9HCGAkUV4PqNxs1t3NCJKtOJkroCmAGY9Q0I2d4BmAGYSVvUxznaWLeN1XoAmAGYbeK96ON9n3scmyjXwccJYAZgdnKAtk8cAAAgAElEQVRyQvr/u9/9btHB/SYC5w72JuJGnFCgtAJcvwff0PS8f9ncOzt95B/ADMAMwMwFV5sCeZhhVv4HArSxAGZ9A0L078I6p9/DPvo3c48D9i5S9wDMAMwAzEpjDYQPBcoogA5VpFFDh6qXDiWAGYAZgBmA2VwGlhhAhm1t3wBpbvGhfxfWOQCz8j8OaJsOexepewBmAGYAZuvAjKe0v61o92idMNxnj3YVqZwBusGXuzraJbW9T0/LxYCQPQXQoYo0agBmAGYb+iBB3wBpbvGxvdvUTK6hxqubhbmAq03lEwPIsK2dG8DqO79s71D33Lq3KRswp3hR59w6Z/QAMJsGMGPDysetvWOzzJJh2MnJAe0oRTsHdvll495cY0mmRyM6XS4GZgaALQGScgOzp/vb/QA4ALNONSbnTfy+j7Vx++TlU/VGjzovl94QjdQbZxy/cy/vNL96tfkBmDmD57eunBY6vkCvfSR+ofzwVXpGqdr/uW8Lv+q+185b/2euXHbCHfMMs+9/+TN1nnW9+/K/EbPF/s0Lwu8z9J3/LPz+8z+il4ReznPY9N/56MGf7z0rdFT0cwfigwAHW8LvWfp3x42f+9wW/Rfx5VG2d0MFV4vS9X3nXVT05d9qZqi1+f3wt2SdVPTSlcvOV0J1mzKnwVxbXl17p0jaNNdP2sLLdO5MYwdNPTt/wdF0rG2sTndbO5v089pYfveUOkOH1RYLfQOkIcf3F//WtXe/+PvN/mZtfn/z+2cdW/jv/6x5jjUfa91L1q2gTjZ1ivPKzzr9vqretb3/c/NzbZpr71iLWB9u0XNcDmM8ct3h90eOK1y/5eodvpDZfCX0k/dce3fpVsrvLB0K0Nj2HJdXzjFyLKxm0//jPdpSO3RQ7UvGYOx4z3ZQAcxi8q3jthiYLRs6gNmyis33fjYwY2zUPn37FTp36hX6pOoE2YaMG7DzdEmdog/ergCavre+bvOz93NHAccL9NyZV+ktCb/qgeAFek4pqkHYt/Wg/DSd+1BAM+125gV67oy4rwprtMBMQ6+//4/o+9UsNAvPXqB7+trAMgHJzHXld++f05eVope+/K/I5N2/F8CsAWbHr9PZrdfpzyvYZSFYBb8MLBOQzFwzGLtIPyee+y87itTOxTpctneLwNQg/Q9epZfOvErf/56FZBaQvUD39HWb3/cu0JfVafrOQQXX9L3y+nvvmgYQtu5duqR/ABD2zg4KGYy12UILzCRc8/UcZRur29a2drbNr2qXZb4Pn1eknj9f/3A1ZIDVa9r+7HX6ia3X6S/+0sIuC8jO0l193eZnYNmzVEMyc10995fNV+NkGYzmvLVunadLou/n1ivdv1N07uXz9MEpfRQ/lAKYORC/3d5V/bhYH67VTtrnRlPPfDtVsN4BmFVQ7M9ep3Nbr9MnFQizEIzB2EW6JPwOv6RIfeminZnW+lxj70qP6GtgdrATmUWmIdrWHu3F/ARYm8cMMw24tmn/6RHt1jMFdumILPiyHXLtL4tM+ulfIfX9/MfATIbnPh8CsLbwiNz7F4dPTtpl+vx4FG27GeNMmKOZiVZr0uQh5U5P92lb3r+PJZmOoD1c8ABytI2bbOx0Q8dQzJwzPNMQTECyNj90qNwOlZxN9tG7ZAaQPKA0gIwHk7qT5A8a+doea7A2dmDmL9c0s8YsJDPwTMC0HxpIVgE0B57pWWf/ir7z993ZaXNbItk5vxqgKQvJDDwTUOyvTy7Sz1V+fnj+vWzvBgnEKhDWOW0R8FU/K/3MeQXWTBweQPveu4RN/wXklzbPzKD1fgSo/B1bGNi+MLxJtLG6fZTtrGx/V/DrFUpVMGoUcWpIpgQIk2kXfgasCdD2N395kX5RPMf2bhJ1r6XemR9LBUCz+d0BMJO2rMt5YO/a+3D1jwLBcxPawyxjvQMwa2aROVpoEKaepQ/+LPQ3ME0AtLbn2N6VHjpbYFaBsWMBwU5Ojmlva4v2jk8oCtPEvfMBZnGoxNuGGUBUL6GsoBN7EpEFSAzNGEr5gIn9fQAWXrvx+f5h+C5Q8++v0lenv7oW6Y9WRgO/mjTX96TcK0jXALgKGIp46zBwUkwBNjCT6FCZJSANJDO/OprlH2HHqc1Pa1F3BLp0MmZzjzujzB0wNsCMwZjxN7PRJg7MJAiT5was2VlleullCNMsMKtnnGGGWT0TzAdffy1nkclzMwNNAzNvyaZwP7v3Th0u27saLC0LqYZ0v1lmKUFYszzTLsFs/O6ZZdH6+jJ950y4JBPALARcpg0IfhTg+1xbyD8WcP0KZtpOaRNsr511+g4tfu4sIDuTexTwSsKqvs69mWKOTtJPnpu0aWCmiJdzcn10ysiHnGO5TtYtnlHmzyQL+32sA/p3bMe8o2fvFvXhah2957Q7az36Y8Z658Aescxw9u639LJynmEmodlFO1v0va/F9z7znmN7V2zAXAVsgJkGYv7eZdINwEyrZQGU5EcGWEkHvRcXzyKLAiMNhxiQheERSX8PaJnw+FmuFi33R9JLcq+wDuEF+eNo5dGEE/l4Qau7B9hkumTYOC+mABuY0Tdql+OdIwvGFKng18cr1OZXdwRmA8O8jpPMt+kMVfvz1Msxvdlm1f16rwsDzMwvjjz7bMrALIRe9/5xpZWZPfsZeqmaRRYCs18jfS+AWbP/WADKDPh6h/7dliIJvsxSy3p28rN0dksAMwPUqjIQyzF12Gzvxg/M4uDL5ivuZ6GZIiWWdbIOAGYx+xexWwlb6LcXZqCp2P7ZsMffxmrIFW9nbd7a/CzUkHsB6WccENQXjBp8PF+jf7+l6Cf+7dci+oR+d/WSJWELf2JrisAsUrcMyKjyLpb5Nu9Z5JkKDvrvK661jfLsXYc+nNXNe67qCzblYOH4OK8jdWiNejd7MBaFhF+jD7YUnZNQzICw6t3m5ZjBs+FzbAeLDZirgJXd1N/bu+xgh9TOQb35P4CZVisEXAFQksBMntelKMOQ53yDnW3FDM6ZEVYBKK4YzbGBaM79kfSGwEw2uHzehOfnz1zXDbSAXiJtzcwxM6WuXnpZu8d0ATDjCtDbkevPOBuzpiE28MuBYm4H3e5vxnuatfnZMNGBig0gK0hWDQTTM8wumA2wm/184h2q0e5hJpZkGjjmLMEUG/xjhlk9sysOwhaBMutv4JizBNN/LjXD7D1y9j6bEDAz8CsCvjQAC/30EszmAwF27zOxpxmWZEZnFJuNrnn5ufwBoTqPQbGm3bAz0BobOI0ZF2E729YGN37xJXMAZjFgaACYs8yy2cS/zc+GNc0ZZm31Tvdd3f1rud5FYAeAWdTWabvl2jvbZ2vsV7wPFz7X9BvHPqbQ6c9d7wDM5Owxe272KEstufzB++Tub9Y8H3uOx7OlB9DqxINjdikmwxP36M9C4w8DzGlJJsMsXTA+UDJAqugMMwGpIjVjeWDWHl6Qv0icrpML/Bo/4W7gmhcvgFkjVU9nbGDG3LjFGrWwg950ntr8WIdm4NN0AOBWbYrNG/sH0/C5k2UHi1y3nKMYgI4dmC2GZfwRAPlBAP4AAPYw6wLRFsOy98hZrll9JKAOW+x9NpUZZiEQa5ZixvwMIHPgWjgDDTPMXDvvDh5dv7odiOzb0+bHbctYj7F2lvPS5mf3D/W+YF2BixgwmrNbGxBr86s185ZoctvL5TTGY3vdquBYdJ+pps/n57t+TyMgfI5+ob3r1ocLn2tspa/52K5L1DsAswZ4aS1i0CvQKLK/Weo5tnelh89q0ewxDcUW3QNgVhWTM3vKziBTgrAZAFXv1VX510s0wz3EYgCsnqkVqRmx+0X0eo0nKS/+tvDc9EYiDJxis+b0TdLdwrM6Xp6dVqcrCBQOBRRgAzO2xsym13aIYsstP/X3HTAdqqrT3uaHXyDdXyD1oNBfhlkvNfJmUQQAjTtP8V8nxwvM7DJM1TqzTAMx76uY1bXez6z5SqYEaL+WZUZWDYt8eDSqa7sMU7XOLNMzzezssnq5pgZkYhnmtGaYWdAVW1L5w2pvsqifv9eZ+QhAM+NMz0rTf3McKIZ5trZK68hfBq7vabGFr513PwwQG0iOs43VQKKlnW31szAjPvvH+tWgZ/DLJJtZXmXSbJdaanvHX8ps4mnzk+mys8vkUs7J9u90f04sw4zXMQCz2nYlwWCLvXOe8ftwi5+bpL1bs94FMChYZugCpeneb5dTanvHX8qs86oBmViG6c4wa3nuBz1+JXNH2Y39ebZY7AhgpruVEvpYghHMwHKAmb7HwiFuvBpYxeFt0/6R/GKkO/PKBWD8jDvrT4bp3h+m1wVmi8OTX7OsAZd+TP6ZPIs0MaFLuetnGZKZ5Z27dKSvAcykqsXPuU6OsnGTewnUS4Sbz9abX4iEu/y8eJuf1mJxR4Nh0NSPVceo1tEdGJrPkqf86g6X39mymo0WmJmN/YWt4/z/439OPzRfzGz85P5kJr+Of/X1TLHEcxqwy18yucK13IeM9dVHDcPMrLFG4xqWif3O2K4p7+uZ7M57d43qaMBXk2/Oizp/gewm/wk/XqYpdHzpymWSedcNDWzeu3RJ7lEm9NIa8x4/te4801bbOf+5CHAbZRurf0Bqa2fb/MyPT6kN2QHMGiD2Pv2NmRkWeX+/dLHdz3wxs3lOwjIdPtfVUda91rpVQdz6HeXtNnS9snWO885H2f+Drav6rb7dYj3Fj6RWK68P1+G5Uda5RfaOfyBgndRy9a6GQrMFZRUQlHuU1VoqUgaUVVCsdhdfz2x9rrF3pQfO9iuZ4ouXMWC2yG0eM8xKFwXChwL9KcCdidE2btWMsNzpR4eqPAgcLTATgCt3HgDMVoBrS8ycY3snYRHOMcOsD3ufu42aQngONJr9LDM5WyzPOdu7KdSVnHno432fexw5y2syYc0dlBXOP9u70iNoALOEwn0VQCJ6OEOBogpw/Z5Mg5QJoM29s9NH/nPDpimEB2AGYLYJgIc9zMr/QIA21s4qkzoAmOUBYykd0b8L65yuf330b+Yeh3zPcV7Vw8LAaO4z2NjeFR00ExGAWULhvgogET2coUBRBbh+o0FzO1Zz7+z0kf8pAK7ceQAwAzADMCsPr/qwb34caGPdNlbrkQI9cM8D0tC/C+ucrnf+u4nr/DYX9i5S9wDMqCTUY3tXdNAMYJaWt68CSKcAPlCgnAJcv9G4uY0bOlD5O1C+prlh0xTCAzADMAMwK297fFvUxzXaWLeN1XoAjOUBYykd0b8L65yud32873OPA/YuUvcAzADMeG8zvYcZG2gcm404oQW0QB1AHUAdQB1AHUAdGGId0Esyh5gupAnvC+oA6gDqAOoA6gDqQNc6UG6KiQ1ZPX78mB49ekQPHz6kBw8e0P379+no6Iju3btHh4eHdPfuXbpz5w7dvn2bbt26RTdv3qQbN27Q9evX6dq1a3T16lUCMEOF7lqhcR/qCuoA6gDqAOoA6sDm6wCA2ebLAO8BygB1AHUAdQB1AHVgvTrw5MkTKvmPPcwSSJIrbsIbzlBg1Apw/cb06cj06UwfEIC2cW3nvmQhlv8b//03CP/lNGB795sff4vw32igG7FYfYRbvuWhaAfi7QB0KacL7F1j46S9h13LZ9dSWuK9jrzX73yFPsV/MQ3Y3pUelAOYJRTuqwAS0cMZChRVgOs3GrdI4wZgRiXrRaqjNWd3wLJysExry/ZODp5w/i3Txsz5vesj7yVtKcJG+x2rA7B3AGZ92LZYHLH6OHs3wLJisEyDSLZ3RQfN2PQ/LW9fBZBOAXygQDkFuH7PviEDHCsKx2L1K9bJmrsbgBmA2SYAnm5h5v7ulc5/zAbCDaCrZB3g/t0mbMqQ4yz9riP8d3vvT5Z8j7KFDWAGYCY3/S83tN9MyNzgbCZ2xAoFyirA9TtbgwDwhI5CxzqATmW4LALADMBsEwNNvYcZ3sfwfcypCdpYwLG+6wD37zZhU4YcZ873GmHF7WbfdX0U8QGYAZgBmC0JNY52SW3v09MlH8Pt6yjwlPa3q40EoX0tJHeoRtHYdAQxyMs4BiboaIYdTQAzALNNDDQBzMJ3Mbd9Qrs0jnZpSuXE/btN2JQhx5n73UZ4of2c0nuULS8AZtMAZmxY+bi1d0wahB3vbdXrQo3fzoFxZ0gmj/ormVP7Yz2y5mukwOzp/jap3SMhhYBQqoJR0v/pPm2zu1IkvUQgS5+adChF2/tLIEetudolmfqlI57gA1y/szUIo4BaO/TBKe8rLM+fx8ywrGW3WOMpdzLfunLatJvPXLnszNxhd/vevUCvfeR2NAHMugOz/+sbVuMf+8ZXgw8lpPzY3g15MLdu2n7+i9a2/eg7X2k+bLD/oujHnaY3/tDd2wfAzH0P47bpMp0747Ub5y807/eHr9Izor/z3LfdMKfexh4+b7U59/JO3ZZ+8vIpUe8UXXoD0GyderCsxlOwd19/x9p5x57pj7b84Wv0o+J9O7vv2rTkcx9/q3lnvfY3/t677/EU7+F+id9fuZS0aQts4UcTXZL5xhnHnvH7pdQZOrx8hRbaOwCzxcDsiy0av/MV+uQfyjblDB0KTbk8Sg/Vm03/j/doS+3QwclJDcx2Duy5hGOxcwCzjsU0MWAWB2FHtCuhlgFW27QM4wrV5DCPzGyxpYEZZpYFkrKBWacTN75nLcxB573k4GWxxlPseF766AI9pxQ9c+WCGVy7HdAL9NyZV+mtqpP+2nlFSg64P3o3AD8AaDGA9jb9lFL0Y994m/7F5/VRArM2vxls+q/B2OdepLOfU1QPMA0sE5DMXL9IPy++FApg1mVQbAeJPgizdozf+wqQf/sFUuo0nfuwCXd87eQS7YMeSJ46Q5dOKaqB2duv0LlTr9An1Q8xdjBpB5aT1iLrD0+iDFbQmPt360L4zTz/Np1V2o69TW9Ie2bsFvtVPwo4No79Ys9ZqDbNvkdja7rnj+1Wor9i+jIxm9ZmC2065vKOG4itf3DvYu8E3MHXMrt/MfTwc4rU596kT995ky79yE/SJ5WOjbsNi+1dMNDN7FADs4MdRRKQ+dcxUMZucwFmPMPJFk4DgVLu5My02qb9/a5LMjUgqu43v6TwDCl/Zhe721oRT4d9ZvfIQic/7dWTzbLF1vh4dheHGamNwYyulnvF40e7KjqLzYVjNizXTQTinbp6cPg6DF12KT04vSl/LxL/0ilzObvOhssvdjDrbdXn/Pg7XnM65tK42Xwuhjnz0kN0yrN19hdr3L1Tt0pHcNPP2A6lC8zcNJlfdQVA03oAkMUAWcrtqxFgxvfG/djebWYA6M6AyJ+Gr5iB5dl9e2RgZmZafO41+noNyPSAUgC0j79FAGbuuxm3TS2DRAPI5IzR8N7ptils6+2xBmZ+W6IHlOoUffB2ifZm6mGupvE07J1rz4zdDKA/2z5pYyPPVTYw/n53sQFTvSfSX2m1aaF98zWdrr0TtqbNpsX8AMwWzzDzNfqFn7Ttxi+EgM3MNhMAje1dx+HvyrdZYKZnl23t0XE1u0yDMA3MOBFKzDxjSCaPswBmBmi4gMqonnInH+5UAKbTbCeGNW58PlQyQIjDW5AOpQFctZLRD8e/dsIlInPtTCfz4U8Ttv8sBTok6qoP2kx+mnDtU76mibCkczCrj9PehG3SXC/bDP19fWTw7rkttwboHdGuKZ8qTKGhG+eqz7mxL3PF7/YsGre6A287n5x3hU58vYQmXz1YrLHfwZrWdaQD6iz/4F923SWbAGYMvLoc41DMahj343c+P6ySA7XNnBsw9sW36Tc/9gaKweDSzsCQS5h0mzGt96/EwNa+01yH5AyyEH6H738+2yoGbHWbtjk3M3PMbGmwAJiZ5UyYYbZKPVhVY66r47Z3nj37+FsU/ggQ3hPYwfoHAyzJDG19aK/abVraFnLYq9TzsT1Tzy6L2eGYvfNhEK4XAjR/FlkzM+9NuqQUnfuHX6rDYHu3zBh4lXsNMNNwjPcukyCMz81+Zh5QYz99nA8wkzOGKrkN2Em5u8CLAniTKjILUARf0dSKtgX0sk/q+yrwk0pHBaycsGQ6FoUbBWZuuiX8CYEZkQZODURyn22u3DzHwukM35pAdeTehxYsvHL0IKHjIr1k2P65jqsGb8LTaOzVBRnnqs+JKJY9ZQMztoYqZ3qxVKT8YCumMXespnkMO6Amn+ZX2+pHKG85pvYHMOsCyvieOBSbJTAz+/nwMstw8Mj7mll7f5p+9HOKAMzWg2p23x87qywcXL5Lesm1nGGas80aTFhmFgVDsDZg1uZXvv0ZjF6xgfUitzU05v7d9IHZt0jbOJ5Va/Mb2kHWYZp9jnXsWdhf6WLTWEdpC9lt1O/confS+J83wCa+tUvC3gGQ1XCrAV/hzLHGz0KxS18U98g9zswyzcaP7d2y4+Bl71cnJwe0s2AG2cnJMe1tbdHecXxPs1kAM61sBaV04TgAKOYegyABvEkVlwuPzF0iDq4c9tjMlIqnLwKIZDo6hGvglUuYvIQ36Q1Bl43f0ct7mi+bmVyRNFsRMuxhFgu7ST9DOSe7Ui9ObOyYuk+7ByBNpGPV52Jp6OjGdWj6jVtbp7yt4Wt7Dn7d602oMXespnkMO6B+PmOdTAAzhmFdjgBmcmDYALD0QNHejxlm/ru42rWdJar3NAsHl+H7391WjqVdsYPCZsCYGCRevkJmJobYz2x6WpQqs/U05v4dg6JxHkN7hhlm68Cx2LOhvepi0xq72dhCdpv6O25+BE7YtKS9AzBbCpj5Sy4bkGYhmf0AQLPxP9u7jsPflW9TJwc7pFq+gGlnkrVDtdkAs1pmCVhqRyIza6iabWZAlDerKAVGZBDmPBJ+LLzgOXaQzwswU3uLWVcdwl0IzEwYFbgL4FAkfk6Hf+S08NH377q8Uz4XaB5Lj9aLwWPEPwhDRiDOg7xXftH8iDhXfU5EvewpG5ipN26t+YvtNdDpF6ZSneQJhhvRmDtW0zyGHdAgn+YLVO7G4ABmXUAZ3wNgJgEY23Ln6OxdVi0VDZZoYg+z4N10lk/HBpjvkv2CXPX+tu73Y59vbYNG2d7YH0Gc+sZfLRQDyeTgcZR57rttXk9jLptxgjJe2h4Cs98MbJi9p/nRQD8beQ57mCWW3kf6Kx1sWm03I32Z6dk7+e6HPwBzflvtHYDZEsAsMrvM18/b34zt3bLj4GXvV/HN/Q9oR0A0syRTXMvlmLNZkukoG4Eqxl+6W2hVz6wy0ER5ywOdQMWFBF7sbMOuw2Pn6FGmQ55XNzsAaHG4/qyxo12GSzY8MzOM91KT0FB7p2BQS7q3t7e9DwDwzYvTynfWRyev2tWGIfd0c/O3SK865MiJV+Zai7Y9zDzNmrLt+lwkCR2d2MCwsZ/D8fB5d+Ph1gYOnfqV9jfronHd2eoyOB3dPZEOqO5UimWYmGHG4GvVI4BZfCCcHij+5sf8Bbnq63LVABKb/iegmLA7r5134bb5ym390Q5vhkUw2Hx3JTs6rvbYn2Fmr5WAZ+PKjxwgD+V8OY25fxe3Ewykhn6M2TNvlmwA0ADMlutfRfor1Re/668CC5vWbgun+gNBYwNiW4x8ermDvfOBD66TAM2fPWZml2lAJpZh+vewves4/F35NrWjYkst9Ywysel/y/5lswFmBvwITXjNXspdFwlDMv7ypL6uIUlbmcWAmQnQ+5qlAHDJdHQBQAyRRP5kOkU+DNTx45L3Bvl24Vpbru2j2+ZjEyyvvd/qwS8FHxvA1BKqTquTPtZDhilnArK/CDMIQ/j5p0IrCeV49iGn3U2TX1ekZjKdorz9eJe85nTMqiNrNuMUdRwd+vyDuQ4aL9ehWzygHUZ4duDM7xUf7V5GtlPKbnLTcE47Zph1gWdv00/Jfkl1/mPf+Crd+O9tfr9Rf8Bo3APIRQNcb4Bp9jdr7J27z48NC8Csg30xA8ZGR1XDsupZM8uC/V24pt/v6bexHszx2wB+Z80HAprB5/R1yZnX5TTmtmac9s5CMc4DH2v75dg1+dXfBc99jE3/ub9xqYJirC0f670XUzZtkS2ctL2zs8uCrwF3sXcAZElA5i63DDf0t/5fog9+hNtYfTxFH4ivZ3L9XXIYvPTt9iuZ4uuY/uyxLtfzW5K5tM54YDAKRIDYYNLWX0LYwKDTmrNTi7C61Kem09ZhsCpmekz5OQCzLsBs9XvY3o1zALkIlK3uD2BW3gZ1sYm4B21nzjoAexe3iVPuQwwlbznr8WTCAjDrCMyajfxdkNbuzvau9AgawCyhcF8FkIgezsUUADDT0nL9nkyDhCWUo5nJMJSO3ZDSAWC2Ogzroh3bOwAzdyAJYAZghj7A9GAd7J1r59juD6nNn2paYE8i9gTADMCMZ59hhtmydMdbZsdT1qtjp6WGy0a58ftL5HmVMNcBZqvEt3HhowngDhUat0jjBvhWFL5NtaO4Tr66QB/cszpUY3vHAycc7YBSNw7r1Fs8uxi4oY1FG9t3HYC9AzDblG3uu66PIj4AMwAzALMoi4AjFBi8AtyhGkVjA4BVFGD1XQc21ZEbcryAYavDsC7asb0DKHMHkrqhGvJ7MYW09W1fER8AHeyda+fY7k/Bngw9D7A/EfsDYDYNYPb48WN69OgRPXz4kB48eED379+no6MjunfvHh0eHtLdu3fpzp07dPv2bbp16xbdvHmTbty4QdevX6dr167R1atXacozzJ48eUL4hwZTqwPcofr0l64Q/oUGgHPF4dzv/vm3CP+uBkPvBI89fWzvfvi9dwn/jQYAZotniK1b9zGADAeQ/+vkPcJ/OQ3Y3v3d7V8l/DcarPsu4/nF9hL2LrR3/+//9x7hv5wGbO9Kj9Oxh1liLhAXQMIbzlBg1Apw/QYsE7BMw0MAs+IaAJa5sEzrgY744o74OhqxvQMsa2CZ1gJ7mJWtd7rOok0J21XAsnKwTGvL9k+m/wAAACAASURBVA6wrIFlWot12hA8281Wwt6F9g6wrBws09qyvSs9KAcwSyjcVwEkooczFCiqANdvADMAs747OABmAGZ9Dz7Y3gGYAZj1Xff6tq9jiA/ADMBsEzCv73d/jvGNwf70nUYAMwAzwh5mRZkGAocCxRTgASSAGYBZ350HADMAs74HEmzvAMwAzPque33b1zHEB2AGYAZg1m3GVt/2at34xmB/+k4jgBmAGYBZMZyBgKFAWQV4AAlgBmDWd+cBwAzAbN1O+bLPs70DMAMwW7burHt/3/Z1DPEBmAGYAZgBmI3BVuVII4AZgBmA2bJM42iX1PY+PV32ubXvf0r724p2j7oHdLSrSC3zQPegcecAFOABJIAZgFmODsEyYQCYAZitCyGWfZ7tHYAZgNmydWfd+5exjXO5F8AMwAzADMBsLvYOwGwiwIw7knzc2juuQdjJwU69mZra2qPjk5PGT5xP+SuZWdnGgIGZAWQC5vUNzJ7ub2cDdCYspeq6C+4X1mJ+32cHzN44U9cLpU7RB/8CwCx7p8XRWNG5l3ecja/HDMy+9fXTov4oevl3GviV9PudF51n+N1T6kX6V9UXQ9cdlE/p+f+/vfN/teM47//8F1axTOgvEfg6xZKvw/W1S6GOI2KrJW2tVpZvlRS5oqoSHOfTJrTCbm6T1IXazSWFQFGVCvkKUaGqXJReBEKKqlaBQPtLsJtSQSk03EJ/+Pzs4A99PsyXZ+eZL7tn7zmze87ueQuu9pyZ3dmd9z77zMzrPDN7/Fyo8aG33cDi7cM1Oh6mo3fPU3icTWNdWPOhArN/izT57T/14CvMO0w35ZtA/zTU7JlzZ4O3hOqWgTVa9m1oP4oqu7vrB7ZHj9l+xSPnzla6hceFdqc1Le5fe3wxzQfPHgieudMv+QWtw7wV2uHrivw/P3tK+X0AzDww+5fNxwKNP7ddl7dK1+TbRbdXxXGP0Z888Mex5vOAUiXO+ePPPyrqpui1P/SL99fnnaF3P+77/UaDTx0P3hK67D5O1j/0W6G/C/NSn8b50g9y2cvh7xRJX8h13nnc2l/c5wUw88DsH74W+ruTl1vkXZa+Tj7jq/TXc1n0/8EmraoN2mYQpmFZAyTj9cv0FsAsBRLZlAUGZvH1DhaYPdyidQH+LDw7Q/sIroulGOV37lAtFTAzHXkBycz3FdrRb8fkP+70YzvlQO8YndYg8qQbWJ18jtbk97PnaLDA7O+O0s/+3FF6y0EuC8gc9GrKc/vLev/OrypSv3qy0oI7m0u/fe8FemTlBTruIIXtmKcddtbJAIxjJ+j03RN0SBzn0y3sYH83SGC2/QI9s/IC/ZsDYRaQMRg7Qb8t8m5qoHPshINiJ+i31UH6820H13Q58vv3z5u2jbVc6m0bu9PAduUwHVpR5AeKzXanNeXB1OC22ncfeI4+cG2hBWQMvY7RaZFnBoqPH6uta5wPYObg1oMX6enVF+lfHAiz8IzB2Cn6nMi7tqFIbZwio52BZQKSme98nH9rXAl41XsZ336envn48/TjaxaSWUD2JN3U35vyrllgJuFafO1L7eME+D/d6O+afNoJOqS0/ztBa4Ef9D8qDM7PcV+/yd815fHxejxxYIVOH0h/JAYwc1DsH1+kI0+9SP/wX/a7hWcWev1nU57bX+r4168qUq+eIp3G/buuB+vVWzK3NxRtbHME2QPaXBXwjCFazXZZgFkYubROW25uZV06aXhTRTqt09ZW2ymZegqlLn+XzlTHa+hjp1Za4/Dnt0Yi8zSBlZCIp2TK8sLjY0AWf6fg3Psvv/74+LoVrbOwNdZfd23Z48w9COtaUyyF90tOYY2vUWpL0x9XeyHdZ7CDqUARA6MRbz947gAp3cmv6ujgjowy48YP29rBT2OHyAAyHlRpaBYBtCEDsxh8aUimDtKv/Z2PMqug2D7z0JH3He5AC92xVwdp7b1MfkOeAW0CoLG/GyQwkxFj+nMGfHG9DExjgGb2Y7CmoVkE0ADMqiixwOb0wDKxrbNmgHjobbv1wCy0y9judLmN/nJI7Yzx7eLHEHHtBqYJgBbUOXMcgJmPBgu00ABNCRAmIsoMTHMATX62x5+iz4nj2N/FwGiQ3zUkU4/Su9/2UWZVPYI8ALPEj0lI1vQ58Xfer+V82um79X4wePaFjxhcesZvVXVI8jbonQM66sxuEWHmo8Yk5Eo+a0imHqO3/jGz/z7y2N91PXK2wExHl8loMv19Y5M2V33om4dpDNX8dimAmQEvESjRd6cu3QEmD3EcrBLRT/U3lwENn4+/e4hjIF1VlssXcw/DyCo+3oOjMJ8ohlCTvmfPr3z5+ztey7iPKZk6Uk8CwSYoFu9bK7q9P8H9MvpO0nba42ovpJcMdjDLBMw+TCLKNMxxodUM0YbcwC/ItduwdA3N8p2HCirFAGpo381USz+tMqhXQ14cXaaPm7qj29QJHkOemYaZjzCLo8i8hvwruJ82x/6OwdKgt2aapQRhPD1TAzFFctqliThTet+z9OcrYZ7W4Lvf/S5sL/ecRHZnBo0mkrF+oGiiHE30hbe7UQEzbj+Tdsa2o/EgkQeXcXSZTg8gkYBCS58eRYp5PTQQU/T05hsiwsxHlP3fPZvP0znZ31VgyUVrDfL7Hz5JSrkIs7geQV48JTOFbL6N8EAIaefpdOTvvCZpW2rz6v0gP/eD39b6u3NUjSWcLzQ/GJgI23yfNwFFmYippdzHTLV0EWaxJg15MrpM68b+ruvBswFmOrost3aZh2TbtKFWafOBh2RLNyXTQBkPrKob05jOwMvt3XpKpoU0gn+lQEmCoCy00yCHAVZaHpHMnwDMskBKHp8pX9Z14vH7BGbm2v29COFddWc0zTQvO/AQTOZFn6WeMmuSttMeJ88xh8/sYJYKmP3uOdpZ8T8C6DXM1syvQpiSWbpzY6GZshF90QArAEtDg2TV9X6Bfu3nFP3s//lCNa3S16sp7yQ9q8K1zwDM6gYv9Z1yhhPBOlOm0++ebwM3fLns7wYNykykWQZ8yXXKqumYDNHOk4VmihRHnomINQAzbyN+kBjZnYm+YGgb5WnY1mB3uszSvnU+5WUGgmZA6Z632umY4kcp0Q54EFQTabWUEO0N+pNVAcW0BnKdMp6O6bQxUzSrGSiP0dOrisYHzCwEe+bzZ4K1yCz4a8r7KgVTOR1o88947rlf1rT9+7TxR5hl/F3lv6I8E23GsyqiPHfMUsKwGIAl39+gt55SdORrb5hplaFGTXmn6KRSJNc+4/5d10NptbenYVg0/TKzflkC1cT0zKWIMNN3wsAT20EIIEwuPQdSJERqvLMpgEoisGT58nNVrixDfuYdbGQUQ7nGiDBRPzZMu20AcrKuE4/fLzCTgC9XN1tHU6cqCo/rXbOV1yt3maTttMfJc8zhM9/HZQNmYX1FZx4RZoUGdkLTs+fIrnsTTuPxYCkzlbECUoudZ6LExHpmsk5NeWbds8xx6MingxUTQSamVUqN8tNEfBkmXzHkOF/9Ajl0YGbgVwZ8cb2S9c2UIn5BgM0Ta5ohwiwbXRfanR1MejCbGVyKCLXY7sYCzMwPIHVTLis/z4PG6MUAmeMAzFJQaACYWLMs1ihc3yw+fpwRZjc/pUiJ9cxkhFxTnt3vOL2mwhcGyDYEn217Gfo734ayPjmfNnZg1uTvwjwLyPwLAADMQvCVmWrpwJmJEhPrmcnjmvLMumfRcTye7XoorcybMDe2w7dfxlM09/YoXOMsjDTrA5glwIgsNAnAVUG1mm9ACJv8aUW6gURjijCL6uIrXUVyMXwzWRIk5bQIjt8/MKumwtaUvS9Ypq8lC8YYksZ1F9F10x4X1b/vr2zfIUASkVYMkMa85ZBrWcfqVyTf6Z/PL/rDPH+6lk3agZBwaYifm4BYU963/zkfXaY14M4ptm068XaaiIcYaSc/XoOK/R2DpSFuJ8EyUyexvlmwnllNdBoizELbSQeP1tbYfoJtDuZm1gIaevsRDhBr2qVkTR+9X/jjidQhhkHL/n0SLDP6NKxvZiPR/BRNtlMJmIb2uQmINeVV9QzWN7Prn6F9neTvwnyjV8anjRmYNfm7NM/6OH7egq34oUDCIHz+OjUBsaa8//yvNLpM68m6dz2OVnkQphf9Fy8BiN+gKaLL+nlLpgAUUpEaWCJ3mfZz8w2oi2qS6RaeVUDPXKsi1SriSZZja5AAwwDU2P2VIFbhNEWXX03RdIBKXEtjhNnEqY3p9RoAVZVv8ystMjclvN7MDkmSK3M9XvvM1bU6d3JgTUJ0v/S0T1OGK69W22mPq7mMnpLZvpcXmNmGbu25Df+GTA3OAMxm04AhJOtoBlLhK7iHCMnsNduplioTIfbtf27Ks9FywVs1o0g6dOS5o24jePT0QX5TZqxN9hdv3akX0zDjfdjfDRGU/T+3/lhuSqV5AYCYhhlEmMVrnRmY5iPOtBYAZu3tztphFGE2we70McNtU+yPHeZFOezPeav9upiGaSOJwwizXBprseyAzNffTsNUucgyDcjENMz6CLNofbM9P4Cs4FG89tdCf3drkWUjy+rzbn4qXLMsB9XitmR5vze0sy182jiBWYO/c+vxZn0h+0SzTX8g1j4PkIwjzexUSxVFiFl9mvLs8cFbNcUUT+7fdT18VvVrk+mpmrzeT/36ZUsDzAygYj0UVXCqLl3fOYZkRscztKu/twI5FtIIRtO8hpmxEgtu2HBCMKfLW6etXfnWzjBqqhmYmcqY9cCq8nWdqrqk1xsCs0nHh1o1gTVTVfefgWzKr2VmkuP7wTYsxZSFyM/B/eLppnqHJm3Da1cCSk48Tp675898H5cKmL36HK2xPShFCSwDMCsyuDO/wkmdn90Iyh0sMDML+Ys2gOv4qyfp2015Bo7Z6LL8mmeIMKsGLnI9KNZXbysY1rwIMfs1Fb1Zk9MHCczkGmWRJhVMq9LDKZfV+mUuX74QQGuh/1Xai6mFS5c20e5CsObfkukGnkL/+I2uDIkGt5VrlFX1U6TE4tb8XOn1QN85KX9wcj9IRb6fNfDAKJ5auGTf5RplUmMDyhxMq9LF2zNNtJlvi6qXAbj1zfi+DBKYmYX8fd24LupTx+mj/eRlgNvS+bU6n97o75p8Wj7i1vvDAf9A0OTvmvIAzNoDQbOQf+bZfvUU/WdTnoFjNrost+YZ+4iuh9H2LZlRxJhc0L/N5z6mZBqgI9+MyBCjDQiZQsW+bsAUl4ZDoMDMCrB9LxUwk1Mv6z4HjZ8cAOAzD3Zm3Q4WmEVRYSXrgY48A4lutuzvBgnMxCL9pa9fNySwvW5sjnWd1V+O8XgAs27BIPu7QQKzDqPf+JnEtjufN0Z/NWudEGHGEWbdbNnfzTwwnlDAYICZroeFZoJOVhFOE2o5RXZfN2CKS8MhUGBmBdi+AcyiddsAzIJosFk7CrnjS4KmsZSFDnx3HXitLfu70sBp6OXphgS2163t5XzgsqcBmAGYzQPmwdd16+u0vsvu23L1BzDrBpSxrty/m3lgPKGAQQGzCXUpmt3tDYim+FUh1xYGtp2SWLTCC1VYF/p0UeZCibavi2H7BjADMMs18F2mjQVylawHOvLdduTZ3w0dcJW+fqxh1q3dYQCZj8wGMAMwAzDr3vfMo1/RZd9xqGUz2MG2G3DG/bt9DYKn2BnArEa0vm5AzemRDAU6VYDtG8AMwKzvTkhJ0DSWsubRsV2mc7K/Kw2chl4egFn3g9a+/esQzgdgBmAGYNa975lHGz8E/9P3NQKUdQPKWFfu33U6aCYidf/+fbp37x7duXOHbt++Tbdu3aLd3V26efMm7ezs0I0bN+j69et07do1unr1Kl25coUuX75Mly5doosXL9KFCxeojzXMwgX0xbTMYF2zcnLxDXj//fcJf9BgbDbA9t13w7Hw56tb2wzp4dtEZ9Dj7R++RfgLNZhHx3aZzsn+7n//5+uEP68BgFn3g9aP/ucbhL9Qg4XvBwx8aQb2d9A5jHBcpjZvXnX96Te/RPgLNfjRf/wx4a87DdjfdT1OH0iEmX0LY/VmynJcrLYkvgG1OyADCgxYAbZvdKjCDhUi7qKIuxnAWJ2WgGUhLNN6zKtzuyznZX8HWOZhmdYCwAzAbB4wD/2OqN9RGNCxv4POoc7L0t7Ns56AZSEs03oAlnUHy7S27O+6HpIPBJjp9afO0G7Xaojy+7oB4pT4CAV6U4DtGx2qsENVB3mQXg6kAZgBmPXdoWd/B2AGYNa37c0DSC36OdHviPodAGa9LBbf97O/jOcDMAMw6xsQcv+u6wH0QICZjjBbp62HXcvhy+/rBvgz4hMU6E8Btm90XKOOawcRVYBtIWwDMAMw63sgwf4OwCwEZrrF6fteLNv5Fh1ezeP60O+I+h0AZgBmd7uPdu3D9wKYAZgBmO3t0V7NXy9rmO2eIdVjlBl3sPtDGDgTFOhPAbZvdFyjjiuAWbG1yupAIYAZgFkfHXd5DvZ3AGYAZtIu+vg8DyC16OdEvyPqdwCYAZgBmI127bO+AdKynY/7d12PoAcSYaanZMqF/uXnbqZqdnIDNPRb36IeA+Wc/dg14M7sY07r7hlFfa4Z17Who/xQAbZvdFyjjiuAGYDZHF5K0MfAfZnPwf4OwAzArO/nYNHh1TyuD/2OqN8BYAZgBmAGYIaXA0y13hv378JRbvlvik/E29XNB7S3vVEtosbpemvyMlFmvUSYla97Y4lc78ad9pu5wMDMADIB8/oGZg+31osCOlOeUrTe5zze/drDHPdn+x5rx/WDZw8YH7b27EbSEWvKq4uKQno4rVLr8cFzTuPnNgLIxunWxlZoJ4KQQ44w+703Doq28Qh9PgPXPv8r9gedj73xheptoJOO63sQv8jnO35OanyYjsqBxNuHhf4Hae09P40lPE7Robd9Hvu7oQKzh19/TNR7lf5evu3z6qrIU/TM199I3gT696esTcZ5WPTf2wg/E0ePWa0eOXfWT1d97wV6RPxoK20rtLvIXu+eH/QbMn8c2N3TdFO+8fPq05HdfdnV9cv07qr8UVuROnUq0GGs/Y5p6sX9EfZRp18SMO2llUDjoD8T5B2gd07647isaa5nkY7Zedzaka/3Br1zILKtx48FfbxQz1AXfsax9X4v6++CdlZR4AuDvLAN1rqOYUrmztPO7l76vKvP5+mdj0V29/RnfV0/94nwOa2Os9Mzly3iq019v/OK1fPJN91LEb4b9mPYhym1St/5jz+m770p+0CKXv6uf5EA79v1kNpHmD3YpFW1QdsZILbXlLe3R90Ds4f0sOewrE5uwAIDs9jQhgvMbDTi+tYuba0DmMX3lb+zfS9S56jMtRyj00rR2rPHTMfKd7R0Z7Ipz3U2I7gDUJaCsg9/1+n4nNM4AGbH6PSB5+gDp+POiiK1ciwAasMFZifpk584Sr/nIJkBY79ysoJipl5bR0h94gh98hOKPDCbfBw68NyBP0GHVl6g4w6Smc78sRMWXJiOuuigm+8OUGigIY6zEMPDC/Z3wwRmp+jM6ov00EEyA79OnXJQ7BSdUY/Rt37oIsd++CI9I7/rYzRQW12lM6spTAMwY7tzW21TK4fp0IocJJ6gQ0p8D+ywwV6dDc8jgqvMOU/Ra6sv0o8dJLupoWsFvk7Ra+oxeveH37AgzNmd/W6B2WtXXZ6EbO5zmbbeA6LBlnfyOVrT7aWLMLOwZ4V2zHfdzgrgo/fl7waWiTzznY87Vw3eB6uLrr+u04EVOn1A9+f4h08LzAKoKKLzjH5Cz7j+aGfb+jvRzpofC9z3wPedp9OyDXb+bvDATMOvj32CXvuYorUKfFlg9trn0vXJfvrNz9Jr6mfonS+7vC//Aq3J73hLZho1puHYkVV6+YiiCphlousMVHvlN+lH3/8MPXnkM/Q9t4+FZxakaTjH/Tse33a1rYDZ9oaije38OmVNeXpds+6BmZyS2c/i/7kbwFFLNs9fR106Pdyi9epXyXXa2mo7JZNfciDrraee2qmV8fmtccg8TW7lVFWekinL89evj48BWfydgnPvv/z64+Prngy56q4tjCaz5YZpEx6j4H4p8lNY42uU2hKF93kfx024nC6z2b7jDsV4vtuOle9oyc51Qx6AWQC3moGh0zEAZiFgM9FmAqDp8oYLzMK1x0zUmABob//wC3TsE4o+uWW3HphNOu4tH8kio6nwmQz4ciBMfrYDHw0yRMde6iU7+XfPVx2qYQKzcBqliTZjgGZAhYw4iwDa/7xB31pVdOaq3SLCLBowSpu5e5bWVnRkot1WURXJoJD3S8tKbXTYEWYSvJloMwZoxu5kxJkEaABmU/ejJBQznz0Esz/4WUiWgqEQrg2/f8dgLO6rcbrsz/HnUIPcPQAwkz6L/Vjk70zb6X9sOn3Xt7Opf/N5rO2wgRmDMbttBcwMIPsE7XyTYVoE0ADMImD2JfrKER0hZre1wExDMvUYfeX7PpKsilyL8tjfdTlm1mVbYKYjyFY36UFddFldntu/e2DmZQjhVAQu/G4zf0pugIEpmfPVpTvA5IGNg1Vi2mP9RTKg4fPxdw9jjA5VWS7fEx6yOsXHe0gW5k8GZjGkyp5f+fLj/ePv4fGaOe1jSmb8AghzD/y5ra5WE69/vdo2x94fv/8unTH6TtJ22uMmXU+3+WzfuU7FONLijhZ3qvS2IQ/ArCAw40i0cMrmOIDZSfqkklFkb5EBaCbirAmYpcdpPbizia3s0Ocie+KOfDj1stIvghzs74YPzDQQCyPF7HRLDc1SKGbgmolGS/O0Fogw8/ZmBoMmmjEcQKaDxDC/sjkzsBSRaA7GSeg03M8aiGm742mX3yATcaY0NLOAzOfFUzJFJBoizILpg0lfK4oUs1MSNTSL+izRfhw9z5FX7O+S8kU01iLnGSBoplpG9XY6cP0UR9zpejnYuCanbEbTNf2z6p/7ZU2r83daDztNU7e1ka+L2lUL08I2eMjA7F9f+hlSZqplHph5uxMRZd/8EtkpnBqaxcdhSmYFuWR0mI4a+49mYFZFl2Uiz370XT19c04RZjqCrG59sqY8fmtmn8BMYgQLfXheMcMhucf0n/nBqEpw0UeCSdmsxvTomlpPybSQRp4rAUoSGmWhnQY5DJHS8ohk/gRglgVS8vhM+bKuE4/fJzAz114HD/mO2WvyAIzTa7ZST7nLJG2nPU6eYw6f2b4XudM027XFHS0As+ZosTAyrN2+TuM4wsx05p1fjqZjDj7CTE+55KhhOR3zb4/SxxSvaZYBZnXHuemdy9ppz9bbdMqd/fB0TAcfeL0Vew8O0iMmIige/ESd/DFEmMl1yqrpmD7yjNcoUxx5pqdiBtFnAGZZW+MIsyCqIrSfFJjZAWUYgZa3V33O4UKyb9BHcp2yajqmn2ppoZkibXc8dTOur10HTUajfaMZGg0E6szWP5H9Ef6c77PwOl4qmmpYpZv26ABpUDQKYBZE1uU1Ye2DKaym3yGmqZpyvCb6mEYfwL5gGbYN/o41qtpasdSBzqvSjd2lbfBggVkQKZYHX1w3A9aUjCpjaKZIfewX6F+raDMAswCYmcgwBl1NwOw36WUVrlPmy0mP4z5510Nptbe3TRt1a5c15vnpm70CMwMwGJIxNLFRPuE0xNmky94Ace4AxOTScyBFQqTGy0sBVCMwy53LRbhZ6JaWZ4EZ69cGmHnNWRvVBORkXYU+/lhdHgO9/QIzeb25umlxpwBmVcSeuDmTtJX1FIfRpOPkvnP4zPeBOx7j2zZ1tBryEGFWMMKMXwwQLvw/jggzF1FmIBlPxeSplxlgJl4OYF8AwHDNHsOdVGxD+GVghZJRZTLfRqDJBdirDn3UyWd/N/wIs6+TfQEAT8O0EWdnrlp4ZvP0mmYWkHH6/2aiz7QW+h9szgIyb0eTgFmYL/XL2WsMkIb6PQRfNuKM1ymzeWkkma1ruK9OG19/g6HX9FsDwAIoZiO0GYJZOCSAUAAWw33Z3w1PZ9s34zo3zgYw9Rf1TqLu0n6efFaX93Ozv4ujxqxPq1n6wEXVet851EX/LSDza5Q1AzO7bpkiu7+egsmfv0QWpoURaB72ZKYX5iKoRplmQZdfrD8FX6yTWaNMrFnG6XprIs+iPPZ3XQ+llXkj5sY2cbRYsNVvy6zLE9M3uwdmFn6wKEqGXnWkEJ8rX7wFdOlliHQDicYUYRbVJRAmA60kSMppERy/f2Bm1w07Q7u1ZU8BzIJ139wFZssX0XVZMMbrmsWaieOi+vf9le17eB2qth3StLPk69qQB2BWFJh9+KpbqPhVH8E2FmD2tokqO0jH/tZOs+RnKtgGa5w5oFYdx4ANUzJrBy/RWmTBfsn0EPfrdwTL9DF8T8YAzGzkmF3oP1jPzLwUgCPJXjRTN7newVZEoel2J9B0GaIrkjpa8BpoxFGk2pYSO4sHnALiZux1qIAsuW6xsH+wnpmZZhlPy/RRaB+J47hM3xa3bc/HvV8Ky85Ruk5ZQ78lgkVsy8PT2QIwvv5gG8BEZw9uGqZ5Q6j8bGBaqhd8nfZVzf7uqH5LddCG1v9AMJ5F/y30CuyN24BMxNhPxcL+BpAF+6SwTQKf5f1so8ayGgcArD66LAfLtJ5cZtfjaNW0oH9TngRr3QMzDRpi+NCtNM03IAOIzOXIdAvPqkg0A14UqVwUU1IVWY7NbIww4wX5BcEz+1fnsuUlEV1VvozYsucL1xyzx1d1aXG9JsKqKn/S8Q6YVfsnJ8gkuDLX69Y+m3zOsNDofulpn+Z6bDkS0obaTntcePa+v7F9D69D1bbjnHaWfF0b8gDMZgNmGpCJaZhm0X81kggzDbrENMxcpJiFgVGEWYvj0JF30EEDBzENMxexY7WK1jdza62EHX0PMtjfDRKYaeAgpmEGEWZmqiZHm/E0TL3Iv5+uaevMIO0N93ZNm6/b4ZmAEQAAIABJREFUHdietxOrRTxAjCIZJUBrYa8MiQa3dXbH1x1EmJmpmmKapYFiinTE2c1TYaSZmbYZTdn0bXHb9nys+9m+SDzd0ugTQbAPM1MMrY4WMskXHLG/G77OYV9t5/Ewwi4EjeG+5i2bco0zTMms8fWRv5P+Tf+4YH4ECNcpy7fB1o/ytMVhb0PotfN0GDFm1ixjSKbfqimnZxqY5iPOtA7LC8maIuryEWbxGzCtdnZfFYA1Xzb7u67H0WpDrdLmAz+90oMwPVWzLi/cv3tgZoFOxYMYPokpfaWFSm6AiSQS0xL5YurS9QVV16mPc9FQraCQhTR8CltUBIaSyCYLbvi6QzCny1unrV351s4QQIaALAVoPMWxKl/T96ou6fWGwMzUgLbWhX7B8aFW9WAuvMsGXCk/rdTmRjq4XwlalRncLz9dlKevVnWv6u2uZ9rjwur0+o3rMvwOVdyRzv9CaTuTTXmuHACzFsCsRkezlpkbAPCvc7rDKqLLhr2GmQVh/OwopaPLfJSYj5yLgJl7e2bTcYAWDC1s511qtfaey3Mdd86r1pDSnXrTyY/aF22DDr7xMYMEZm46JddBKRtdxnWp1i9zz1z8JswmYIZF/9nu5DYaQIpBo70HcnpSg726SDYGTsPbNi/eX61fVtmdeyGAXPdM50WwTOswvn5H3A9p+d1AsYzfcovVh+uUKaqgmINn7BOqdDdNk9OHr3MOggm94qizSBc/tdPeD7Sz0s/x59TfheuUiReZNLXBzt8NG5Txmy5DYPZTA8WE3TEsc2uV2UX/fb5/uybWMKuHhTlgZqPLkjdnmkX+vb7s35R5eUCPEWYekIUQbD/p3QMzDUE84NFwxwAQDY1ieFEIO/ANKVQcioECC6UA2/fwO1QtO6bBeh8NxwCYtQBmfnplu5cDhPt7sJSDTcuZho48d9672bK/Y8iErY0wAzDrxt7k8zw8UCamU7q3WpauA/odDX2Qtn2Vhv3Y30HnUGf5XOJzN75vHMCMwVmZbT008lFS2Gd6LdjfdT3IVvsBY3X79grMTDQPw7MQpJUUq68bUPKaURYUaKsA2zc6VGGHahoAhGNCIDZJDwCzFAqi895N5511ZX8HUBZO1QQw69butP2Vhk1jKA/9jqjf0QC/ptGK/d00x475GG4PsO3O7wGYpZANMGx6GNZGO/Z3bce/0+43EGDmphQ+jBaHD+DZtBLkj+v2BuSnDfI5W00fzF/2SFK70KeLMocrN9vamDtHU9UNEWaIMBNvtOwL7qED310HXmvL/g7ADMCs72dtDICrdB2mapsLQ6UxXwP7uzHXcZq69f3sL+P5AMwAzNpArpL7sL/rekQ+EGAm17ji6LIInhVWqq8bUPiyURwUaKUA2/c0nY5RHwNgBmAGYFazOHC3UKvLwQX7OwAzALMu7SxXdmnYNIbyRt2HWACwx/4OOoeRfLnnE2ll23UAMwCzkjCsTVns71oNfmfYSd2/f5/u3btHd+7codu3b9OtW7dod3eXbt68STs7O3Tjxg26fv06Xbt2ja5evUpXrlyhy5cv06VLl+jixYt04cIF6n5K5gw1nPJQvgHvv/8+4Q8ajM0G2L7RoQo7VNCjez3QQS3bQYWek/Vkf3f/3zcJf14D3T2C/Uy2n1k0QpvSfZsCjUON2d/NYrdjPPYPHrxJ+OtWAzyL4bNo9Pj9L9KH+OtMA/Z3XY/ThxNhNiX4mvYwvgHTHo/joMAiK8D2jcYt07gtwC/EY74vY+yIo07dQodZ9WV/B1jmYZnWQv+bVVsc32z7Y/blqNti9h/Y3+HZDJ9NwLJuYZnWFz4h4xMAyzqDZRpEsr/resw9HGCm34jpXl8dbv0UzZJi8TlKlomyoMCiKMD2jcYt07gBmHXa6UEnPuzEQ4/u9WB/B2AGYNb384Y2Fm1s3zbA/q5vW1/08wGYAZj1/Sya8wGYAZjxWzO7n5KpF/1XZBfD36Uz61v00Cxrtk7qzG4nDIIbnE4KR6FQYM4KsH3PpfEAkOoUSC36PV30TjWur3uA1bfG7O8AzEJghrdkdm/ri+6PcX3jA3rs7/r2s4t+PgAzALO5+DsAMwCz/oCZfsMhR5J5YEYk08sSCG5wypaK0qDAYijA9j2XxgPADMDsbvcD1UXvvOP6+rMB9ncAZgBmfT93aGPHB6QW/Z6yv+vb1hf9fABmAGZzeXYBzADM+gNmOsJsnbZ0WBnZaDMTWPZwi9YrkFYWRHCDU7RUPa3URccVLXdiYUKzifvaHXbPqM6i91pewhx2szqZez+X+9Rfldm+59J4AJgBmAGYYe2oHm2A/R2AGYBZ34N6tLEAZn3bAPu7vm190c8HYAZg1vezaM4HYDYOYMaOlbermw/IgLAHm7RarRm2Qdt7ezY9s+1+SibRw611NyWTiMR6ZnaaZnnQwHoULXmBgZkBZAIS9Q3M9P0tOb3WlKd4Gm/Lu2jsiiMZWx4z0N3YvufSeMwNmG3QOwdUuBbi48eWGl6Vv/+TNV70TvV013eW1lYi2zp2wkOx916gR6r2VNGht/uLrpquPot4fdNrzP5urMDs6vnHhF97jL5424Gxv1rNp7u3hWJKZgs7f/uw0FA+44fpqIa+Qf5BWnsvLLO8j10EADXZz3/w7AGh2wF65+QiXPeQrmF6jdnfDdn3Hz930NjPI+fO+nb07nnidFtH9wzyjy8T2lkAsxCY/c7rVuNHXz8XvD2U09mODr/rjnv3iHimpS88Qr/u3kA6Tn93jj58aSWo+9qzG37s0JSnxzsAZu00ePkJofEBeuc18XbR136e1kQf+vTLPo/ttOvhuF/03wAyBmMPaHNV0ca2g2TbG6RWN+lBBpZpuNYHMOtaiLj8Tm7AAgOzuP7DBWZ6mq4GZbti3bu4djXf53Z/aq6nw2S279E2blkoZzugp18aUqd5aNc6WeMhd+Lrr93CnDwIO0GHlKKq428G2OnAur7scAC+vPtNrzH7uzECMwPLDn+GrjoIVtXRwLIYnq3Sn4n9AMyme7aOHlOkNBCPn2XzPRzEj7ONbfbzBpYdeI4+yLbDQ2vT5nW902vM/m6YbQW3lyfMj1BVu2mg2Ak6tPICHXeArHoOOW9COwtgxsDsFTqsFD36+iv0qSf0VgCzv/k0PfrEp+l3HACz8MwDsVjDX/9lReqXX6mA2zj93TE6rQT0P/kcrVXfm/Kc7wAwmwzMDCwTkMx8f4J2jHa/RKeVorVfPGXLifZlf9fhkNkUXQGz7Q0ByPa2aUMxPNPQLP4eRpstCzDjqCV7c3iKqI1+4xumlE8nM2WUKfw6bW21nZLJU1At/LFl68gnMWVQnsfcSpmnzykjpWzemV1ZnrhOE7QXTsFMgdls5fNUWq8TX19c7uSosLprC6MNbblhWv3jFN5b1kKXoXWq082eo0nX+jOat1bQuiDm/v0VsSaslSstsCtFrY8TF8P3YZyNW10Ht7kDulxa1Gk0a/pkjYfZiZ80sG6AOckgumFf/qUc2yCqwNpMg24TNGZ/V8EkAY2GnbZBv6QEFBP1SkFauq9uEsb5PE56XmfIN1EsFnibaBcxgD99Vw/2Qxg+znalyc9HA0hAMx+Jsi8tpteY/d2wn23r70NgFj63wfM3oQ3QWsSwB9/PpcDMgbJKGw3Q1EH61N8wbBPbTN4o/Z0BZCu0Uz2/wsc15fH+AGYTgdkHv3iA1KM/Tx9UWmlI5gBaAM90ZNkpeudRRRxlxv5ODHE7+WiBmY4uCyLIMhFmG9tznZLZSe0bCk1uQN16aXXpDm55YOOgi5j2WH96BiYMSvi7hyMG8FRluXxPTswUVg/N+HgPySwg4vL1LFeGRPaqJn3Pnl9AvP0d76CjuP56bXhKrr92CyZ93eyxts5e/8YSudLRGnOTdEvz43rXn9Xag78+fpmFK1NoEd6raY8Lr4Tte5SNGzdSydZ2QLnuqvqFaFZIhOO9HU3WeNid+LDD7utiO/fetvygOejUGxA2eSDgy6073zKmT68x35dhw7Fw/TFTl9ufoSfUY/TEYf5hTpH6jQ0yeSbCTEaUaWCm6Jf+ypejWwXY2v6epSCqJRmk2+gYGWnqfeOY2okGP++iL9bk8gdY+mAKaDa9xuzvhv1sT2onORLNTtls085WECiGQkv7vQUwM9Mw8xFmcXSZ1nec/u4c7Tyu21gNzexzKadkNuUZPSoI5KcRYppmpEUCxWxUmYZiKUyzwIwjztjfhaPc8t8MMNPRZdXaZdW0SwvNzIUEMC2MLluaKZkuqkdwDHs3GtMF1NF7t57yZ6GJPJeBJjJBrreVhXYarDBESsuzbxjl/AnAzJTv97UVn1C+rOvE4/cJzMzbUevgIT8kts4eSHF6w1Zes9ltkm6Z/KSMmvPJ+yd3mXQvpz1OnoOomic+1satTb3sGivyV6MxDWgWoy45jYfdiW83uDadd2WnZqUd+fOkB91Nv5wvg0az1nE/GnOHanTALJ52aQCah2J/9hsCpDmwBmDW7hnO22cKxAxAqyLFD9IjK+EahW3aoqHvE/h5s55PPH1JEZZCmK1N3o/G7O/yNjyL/fd5bA0wM5Da+TWxTmibdhbATESHGUg4CZg15dtpndX6Zg46Dt2XNV2/BWOKVGa6eVMe4FgEx2oA4s7Pyf7KAVpzUWQpMPsi6X3nAMwy0y3NemartPnAwbFgfbMlBWYaOBiYYW9oAGJy6Tmw0RamyDeBOtDRCMxy5wrKyICdCDrFkVHBd1E/bojtliFapnxZ14nH7xeYScCXObfRzKYH9ymCRslXec2iDMkpLWhkWJc5d1JGchabULefTg+m0+rdxXmmPS66DL6PTY3D+PN0WDU68t3e51TjYXfi2w4Y/MA67cjXDAQwDXOf0U7tNWZ/N05gJqPIXqcvHlb0xPnXbZSZmKJ5/9/TCDOsYdb2ebb7pc9yfLy3SfZz3frX2SBMuWsTft4AM/lDVBqRUe68i1L/Pq6jvcbs79gGh7md3E42/2iSHg9gtj9gZiLIxHpmUj+ztlkmb5zPtnj2zp4jC6/5R4GmPOcXagARQFoTSFvECDO9oH803fLB5moScRaucRZCs2VZw8wzBzstLgQpOlekG0g0pgizqC5ejBDocLoEOzkteD+3TYBglJ985TJ5m+5QYNF/Aaqq8vcRWVcdk/mQBWMMZWOtxTmnPS66BO5QjbNxa9l5DRbubHlMMs0TxzXaUEbjYXbe44HxhO9inSO7MLhcCNx25OW0raXQpDQQ3IfG7O9GB8zclMzqrZj/3gDMkimamwRgNuE5Dmw2hWHJc5tM0Tw/2ilKgd+Xfl5+Nu0lgFmg1bR9CKmr/JzRmP1dYp+BPe/H9uexbwq8kvrsow3Qx0rgg88antVHkDXBsj94kI8u05oWsfVpn5GOjjOALIgq8z6tKa/SAsBs4hpmCTyUUzTlZ6PlnNYwy4KwJKJMR6GJiLNq2qYFZ8sHzHIgRRMJmW7hWRXhZMCOitbIiihG9VWWYxMToBSAE7u/EgTP7B+vcVZN0XQRXVW+jNiy5wsizFy9qrpU18kf0usNp5/a/Prj0+vhkuu3rsz1dZL19vtPPqff132SkM8k2TLkixxyugrZ9zHtNrIPDVvN/XDnFIWG55z2uLC23KGqHHpHDc0ilb/zOP8iZCGXCaEOGkHAr1nvVxuNk07voDrx+YHD0WN+zTJdPzNNq1oMPBpoZwbVY9SkdJ1m0Zj93eiAWQzI4imaVYSZjS6LI88AzPLPc852ZTRLLt8u+J9OtZ7Vpy7i8c1+3g8mzbXHUzSXoK9R4p7NojH7u7ydtrf5+R6fAWYakMXTMN3SB/z8VT9EZdpZQLI2EWYWoqlM9Bjr1/TmzBK2v3BlxFGzBli72SlNeezrAMz2Ccyit2L+vo82M2AtAmjs78JRbvlvqhaE6cizal0G+QbNMLpsadYwM4BKzK9loFGXru8VQzKj4xna1d8FpKq/nRaa8ClsUREYCoCZ3sOClOqeBefR5a3T1u6WeCtjGMUUArIUoDEMrMrXdarOkV5vCMxMDUzEV/74UKsmsCY1MyBJ8fRIzol0cDbcqswaYBa+BVPq1qbefF2ZbWAfPL1V7xfVodLZlTHtceIS+D4sXMPEDUwXW9OwiWcYsKz8r4EtNJ5vJ7yjwYJcV0X7nAqWufOZX8LZ9kK4Nko9uoCgM2jM/m58wGyT7rt1y7iO1RplUXoMy7QWAGZt/YGF3sm6g8FzncIy/WyPso2d5OfdgJJtEuuXTfFj3AwaV7p34Yc7L9M+a1wH3tpnz0I0TlPRG2lPB89j2s4y8MHWRoh5HW3f5NHXz9EfmEX+ua8itr/8iovQs8eafTMvSxilvzvLi/57PdJF//N5Rg8As8nA7LWfpzU3Xtd2yeuTVZFnQb57e6bTle1YDHE7+WjfkhlFjGkItp+/5Ysw6+ReoNCFUyADxBbuGqe7IHYwY23cUK8pOuhdQMpMmQBEbQfp2K+UrbC/GyUwq6LI/Nsv29YTwKz7Zwxt0eK2RWO9N+zvSvnPsZQDUBZHmJX/PtZnaqZ6AZhNBmYzaMT+brrRcPujAMxqtOrrBtScHskLoQCA2UyNRAaWoDwMHsbS+UY9uocNpTTm9rwtSFqW/XQzW0pjlJN/HtDmoc3r2wbY3+GZDJ9JALPygCzWtG9bH8T5ZoBBVYQVyqiFbuzvusYGAGY1Cnd7A6IpdyIMUZ+31fTBmuseR3IX+kxT5izAbJrz9Xf32L4H0dgAvI1qWg868WEnHnp0rwf7u2UBYW3rqVsc2F+39oc2FsCsbxtgf4dnO3y2Y7iD7+UBWt+2PojzAXbVwq4SQJD9XdcjaACzGoX7ugE1p0cyFOhUAbbvQTQ2AGYAZp2vmxJ2rDHQGJce7O/agqRl2U83MrD1bm0dbSyAWd82wP4Oz3b4bAOQlQdksaZ92/ogzgdgNg5gdv/+fbp37x7duXOHbt++Tbdu3aLd3V26efMm7ezs0I0bN+j69et07do1unr1Kl25coUuX75Mly5doosXL9KFCxdozGuYvf/++4Q/aDA2G+AO1SAaGwCzUQGzt37wJuEv1AADm3BgU1oP+Ls8tMAaZt3anbZjtLGp7f33T75O+OtOA/i71Ob0c1i6XUF5qf/86PpXCX+hBld/9EeEv+40YH/X9TgdEWY1cTx8A2qykQwFBq0A2zc68/mOFXTpThfAshCWaT3Q8U473iU1gb/LP88AZt3anbZhtCWp7QGWdQfLtLbwd6nNAZh17+u0vwMsC2GZ1gOwrDtYprVlf9f1oBzArEbhvm5AzemRDAU6VYDtG535fMcKunSnC4AZgFlJGNamLPi7/PMMYNb9IBJtSWp7AGYAZvN4Ltq0FdhnNp8IYAZg1jcg5P5dp4NmIgIwq1G4rxtQc3okQ4FOFWD7nkenBedMBxDLpAmAGYBZ34MS+Lu8zwEwm21w2MaOl8m3t60rgBmAWVtbKblfm+cV+8zmEwHMAMwAzPb2aK/mb8xrmHVKLVA4FJiTAhhA5geQJTtnKCuvMYAZgFnfgxL4u/yzCGA22+CwjR2jHUhtD8AMwGwez0Wb5xX7zOYTAcwAzADMamCZhmgAZtNQj4e0ta7ozO40x+aP2T2jSJUsMH+a8qm7Z0itb9HD8iWjxBoFMIBMO/Hz6MAt4zkBzADM+h6UwN/l/Z1uHvq+F8t2vmX08ZPqDGAGYDbJRrrIXzbfM4/6ApgBmI0WmHFHkrermw9sJNmDTVpVyi2mtkHbAGY16GHa5MnAzACwfYCk0sDs4dZ6PwAOwGxaI5r6OH7eu+iULEKZHzx7wPiutWc3gkWXOZ3rf/ql/EByEeqw6NfAWu5X4zEAs5OftW3jwdfP+Td+Xvs0HazaTEVH3g3B2BdeP2hsMjjGvTF0Hh3bRT/n0WNW40fOnU2gTjbvvRfoEaH/obf9L+X8vC/6MzXp+nYet5rIZ46fQ65j7NPC/AP0zknv83QDsuh20Pf15Wzr+Dn77FqND9PRu8623j5cLTjM+sf7TLqnQ8gvbXcAZikwu/KqfbZXv/ZG9QbRH3ztscC+Nt4Tx723KvIeo81/8nlsi0OwrfQaN+idA1YLrod6/Jjvx518jtaEn9+Pv+vblyzu+c7S2kqk8bETVVtQ6++c3+P8XNs8XGB2ht49FGnywvHgJQY//q1HzTP3zG+dyaZbe32SbkZvCu0bIC3u+b5Ip56MNH75ZPVShG99Rbazip7/TviygDD/IJ36ns1nPzH1gLjlgX4NMwPIGIxt04Zapc0Hbhrm9gap1U16UAPNEGHWUu1gt8nALNi9xRcAsxYiYRejADuYtLPiB1PDzDtGp5WitWePmU6XHFh+qDtaB56jD87aOtqB5ArtuO/DrO887tdsGg8emL17hNQTR+jIE4o8/HqFjijxXe+jDtLRaxqacd4rdDQ4xgO1xe1Ye+jU6zVqELFymA6tKEo65dm8E3RIiX0NyDhIa+/Z6x+Fv3tphdSBFTp9QPs390PABJ9mfJzwebGP041Br/eVQdOibutsa+UFOu6u2QA1MbiM9YvzY80H970DuwMw83DLaKHh11OrtPGUogqY/dOLtPrUi/SDn9h9LTxbpSv6u4FlApKZ7y5v8G/JtMAsBmH2ueG+h/N/2jaV/xFgkr+Ln9Xl/W6BmfxRyWtxgg7V+jtuZ08Y4Ja0zYN+S6YFZq+dTyPEPrp+nF5Tip75reMGqoXA7Di9duh5+rGDZDdfUKQi0La4ACsEUt1fpwVmMQgz5/3eUfr4k0fpWz+y12Th2FP0Vfld5Mtr5f5d18P7Cphtbyja2BaAbGNbrFn2gDZXBUCLwNlyADMNuNZp6+Eunal+3ThDu2TBl71hOl/eMpmnqaren/8xMJPlhcenAKypPKJw/8nlU3Dt8vri8yhaDyvGlTBbE4lWaeLrUJdOD7doXe6/hSmZgaA9fGEHM7jOe2u4ZTtd1cAyd5z5pdJ3tsarRVdQbTqNhw3MzhnodeRdu62AmQFkR+ikixh76we8n4dinFYdU+37JqBFAFC4M2+3Yae8Js8AMhH5c5f3Gwsw40HkhGcu8Gl6cNns37CGmQTCbDM5u/P7megKMaD0A83zdNpEOXpQq/OG3a50Y3cAZhKYvUGbTynaeM9uK2DmQFmllQZoykIyA88ETPvvn5yiDZen9x92/45tLtNvMYBM/sgp953s74JnNWhz/PO9HPuwr5tc77y/q/eRQ48wywMzhmgWqoXAjPPs1kShCYCm9ZBwZ7k/NwAzB8YqfTRAUxxFdpKerz6nkI/9XddDZwvMdHSZjCDTEWUBMNujAKgtLTDLQyVeNswAomoKpYNOnElEFiAxNGMoFQMmzo8BWPo9PF+cn5YfArV4f3d91fW77+L6s8Zo4Je/5mqfunQH6TyAc8BQnLcqAx86U4AdzLA785kOVQXGJgws9X5J56upPOSltjKdxkMGZmZa5WdfoRh+mfQnPk1fqCBYBNRMei7NArXl6KRP7pxrHUwH3UTwpJ3yury0Ux8eO3R/ZyInzLSkCc+c9GkOnq3J6U1yatPZcwRg5m2yzrbCZ5MjLNJpwnq/OLpMp6V+czhtSVd2V0GgGAot4XcDv149Rf/9kwnATEaRyc9GMw3MNHSzIG7Y/s76OK5DcwSZ8Ict/F34LPtnf/nSbfvoNQ4hv9ejzt+F7avf/3wwVXFY8CyekvkovfsXIQz76PokYMaRaOGUzQoCxVBo6b7HUzIZiKUQ7Op3niKlXISZg2cfl9M5xVROtuPOBsyuYAPMNAyr1i7TMMxMzxQRZRqgaWfMEWhLDMwkPzLASibotbg4iiwLjDQcYkBmgZY8nEjmR0DLlMfHslk07O/AVFC+XCusRXlJ/fi0cmvKyby8oDE9AmzyumTZ+NyZAuxghtyZb7520ZGqIJocqEzKl/vic17rSRrm8wcLzMwaZRxFFsKvFJi9SXqdszCaLDxG6iA7nEv92UTocKRY1ClvyEuBmYUXHJ02aH9nBoIcVZF/puzzGeVF05XMlHSlSE51AjBzg+YG2zLPo4lgdGuv1E7HtIPLeJpT3ncOoE3p0O4AzFyEmYka46mUTcAszeM1z6xve4xWTZTaGIBZ+GwYaKus/8tNudTr65mZBC383VK3rQ0Rdab9VNzunqfTE/1d1DaLsocFyWIg5r/b9cri9chqgNn5J6uozng6JiLMMjBMTrNkKBbAQwvWPv6VL9roPAPPBFwzAM2vccb9u84GzK5gtben1yvjtcvclEwNxBwkMxeyukmbGwKgAZgZ+RKgJIGZ/FzdRQnJ5GfewUZbMeQKIsIcgGLD8FsP0YL9WwEz1wGspkfq7768uH7me7WvgF7i2nzkmAmpq6ZeVuk5XQDM2AB627L9DLYzn4VgsqMVDR6j/c0ixg1r+4xXF6nRrJ+n01iCouF8trDLL+Qfwq8UmIX5tp65NESY+UGM7YR74CA75U15LiotmCYnjz1fdWaH91zbZ8xDrvpnLvFpMtrM+L/0WAAzDcyabcvbp4VryeDSDRZz0FYfOzyb0+1Ct3YHYKbBloVgHBXG33NTMg0cC6ZgOuBWReSNKcIs7pfYdcu0D0yBmfBpLfxd/CzjO0fZ5WG/1ifv78L2Veo4FmDG65aFUzRrgJlY5D8H2hBhVgfN9FRLD75Yp6++rEjJ9cpktJkBayFQ4/Fs1wNoZcBYNP1yLwJitVDN7bc8a5iFkVQxUCIJgwxEElDJ3EkZEVYHzDywCgBYtrzQPIL9WwGz+PrC8pL6hdmZbyHw8zuI9Fw9AMy8VD19YgczzM583KHKfRcdKcCyjgZt02k8HEgm1x+zC/fzcxNs9VRMrGFWYB0222kPtOUfaFYOBm/ADPbRoGy0a5jZwWJQX9ZEAP8Elmmf56Yo+bdips8rgJkeMDbZnV/svxoUZtYp4zI87OWB6FCBWbd2B2CmgZe2M7LwAAAM0klEQVSFXNlnW8CxybCMXwLAkWpDX8Ms6s9JP1YDxcwPCnK/mh8IqmdYREQhLb/2YqVL1t8tATD7i+fpGRVPy5wMzD7KHMcgCNsInAXrlNm8BJZpQJbsNydg1rQ2GYOzZMpmBNQAzBzlkMDMASvF4WK8hlm1VpcFZklEV5UfTcl05VWRWhmwsi9g1qI8A8zE9WROGSXlIKDeRaZbeFbVg6PT9nWe6LT4um8FuJO2XMDMDhiVGGiOt/5RpzOChmXqnQ7AOTKhSeNhAjMJz/TnOFrMArUqAi0BaLljfJlV5xQdeQHe6jvlHBXEUy4TYBEBtPH4u/iZa/Jp0b7xlKWz50y7AdvzcMtqEdmdHjCKaZi5iItcGutaxtf24c+bzhHZkotAy/v5aN+M3QGYxRFiPuLMR5jZCDQl4FleNwve/HHDBmY7j4cvKgl/DPDRZua5CgDaZLvjZ3LZt0ePhWuWmbUXOUK7hb9L21/vQ4caYXbzhRCOmbddRov3Z9cw04BMvBUTEWYRFBNTLb/6sphW+aM/ohCOufXNZGRZdWwIyOz6Zr4s7t/teyC8zwPUhspNtdRvxfTT9YL1zSJYpqEagJlTPQBmOs3CIb6ZKoBCGiKt09aufGNkGPEVAjBdHkM2f29kmeH+ElKJ60uuQZSlf7mW+QyzVMNbMk2dRRkMCOvSTTWiOuvzyPO6y8WmOwXYJsfRmZcd/fwv436NC2GrHKkRLYQ9Pk2kPiU+z6bxOIHZm/SWWeOM7esgHb3GQCwfnSbXN1v2Dny+/hG4CGBiJs/8Eu71X3vPd+LH4+9yg0Kus9iyTzNRFz7dT+20fkC3MHntvXbLlx/blv3ONqRUONhkWOvhbajdONqTsnaXBz85iLRMadE6ZWZhf//sVvanXxBg1j7zeRKWaW1530HanoFgvm4JlA18WgjXbFStPzb2d8vny0JfVNVfrlGm+8EMy0wb2+Tv8tG40vcNFZh9JNch05oEsMwu5s/PFW/t2zJt1BmnqSQqDW/JrCLrzNRK/3ym0y5FHo/PeHF/t24Z6/z8dzyY47TuRsy2ZPuWzAwE4+iyNtvlAGZd3wqUDwX6U4AdzCA7VJ1ES5UASSijjT2NA5gxDCuzrTqyARSq6exin31DHvi7vG/SLQ5sr9vnrI1PXLZ9AMy6BYHwd3l/B1/Xra/T+g4WmIl1yErXoQJGVcSUhz3Im10L9nddj6ABzGoU7usG1JweyVCgUwXYvpeto4765juSfeoCYJZCNnTku+3Iw9/ln3usYdat3ennuk/fOpRzAZgBmM3DVtHOdu/vSsOmMZQHKDY7FGvSkPt3nQ6aiQjArEbhvm5AzemRDAU6VYDtex6dFpwzP3hdFl0AzADM+h64wN/lfQ6AWfcDyGXx6/upJ4AZgNl+7KXUvn23O8t4vjEArtJ1aII9yJsdpnH/rtNBM4BZvbx93YD6K0AOFOhOAbbvUh0RlJMfkEKXVBcAMwCzvgcS8Hfpc6h9E4AZgNk82igAMwCzedhd3+3OMp6vNGwaQ3mAYrNDsSYNuX/X3YjZllwswowvGNvMonW8eB221WKksBPYCWwANgAbgA3ABuZnAxqYQf/56Q/toT1sADYAG4ANwAZmt4HOgdn9+/fp3r17dOfOHbp9+zbdunWLdnd36ebNm7Szs0M3btyg69ev07Vr1+jq1at05coVunz5Ml26dIkuXrxIFy5cMG/JxM2e/WZDQ2gIG4ANwAZgA7AB2EAfNgBgBjvrw85wDtgZbAA2ABuADXRpA++//z51+VcswqxrsjeP8rXw+AcFxqoA7Du9s9Ak1aR0CjROFYUmqSalU6BxaUVRXhsFYHepStAk1aR0CjROFYUmqSalU6Bxqig0STUpndKHxgBmDXetjxvQcHpkQYFOFYB9p/JCk1ST0inQOFUUmqSalE6BxqUVRXltFIDdpSpBk1ST0inQOFUUmqSalE6Bxqmi0CTVpHRKHxoDmDXctT5uQMPpkQUFOlUA9p3KC01STUqnQONUUWiSalI6BRqXVhTltVEAdpeqBE1STUqnQONUUWiSalI6BRqnikKTVJPSKX1oDGDWcNf6uAENp0cWFOhUAdh3Ki80STUpnQKNU0WhSapJ6RRonCqq1zDDv24VgN2l+kKTVJPSKdA4VRSapJqUToHGqaLQJNWkdEofGgOYNdy1Pm5Aw+mRBQU6VQD2ncoLTVJNSqdA41RRaJJqUjoFGqeKApilmpROgd2likKTVJPSKdA4VRSapJqUToHGqaLQJNWkdEofGqtVxW9t2KDtvT3aq/62aaPKW6XNBzIv/PyXf/mXpeu+EOX1cQMWoqK4iKVUAPad3nZokmpSOgUap4pCk1ST0inQOFUUwCzVpHQK7C5VFJqkmpROgcapotAk1aR0CjROFYUmqSalU/rQWG1sO/i1vUFqdZMeGGD2gDZXFQV5KgZqHpoBmJW+9SgPCnSvQB8OpvtalD0DNCmrZ640aJyqAk1STUqnQONUUQCzVJPSKbC7VFFokmpSOgUap4pCk1ST0inQOFUUmqSalE7pQ2Plo8p0RJmDYg82abWCZxqMRQCtikKz0AzArPStR3lQoHsF+nAw3dei7BmgSVk9c6VB41QVaJJqUjoFGqeKApilmpROgd2likKTVJPSKdA4VRSapJqUToHGqaLQJNWkdEofGocRZhvbdkqmjjbjzw6ObW+IiDMAs9L3GuVBgd4V6MPB9F6pGU8ITWYUsMXh0DgVCZqkmpROgcalFUV5bRSA3aUqQZNUk9Ip0DhVFJqkmpROgcapotAk1aR0Sh8aKz31Uum1ykRE2YPNVVrdfCDWM9ujXBqvd4YIs9K3HuVBge4V6MPBdF+LsmeAJmX1zJUGjVNVoEmqSekUaFxaUZTXRgHYXaoSNEk1KZ0CjVNFoUmqSekUaJwqCk1STUqn9KGxqhbz19MweUomIszMvezjBpQ2GpQHBdoqAPtOlYImqSalU6Bxqig0STUpnQKNSyuK8tooALtLVYImqSalU6Bxqig0STUpnQKNU0WhSapJ6ZQ+NFYcJaa31bTL4AUAvIZZ/ZsyEWFW+tajPCjQvQJ9OJjua1H2DNCkrJ650qBxqgo0STUpnQKNU0WxhlmqSekU2F2qKDRJNSmdAo1TRaFJqknpFGicKgpNUk1Kp/ShcbToP0OxaJH/BKD5N2Rq0AZgVvrWozwo0L0CfTiY7mtR9gzQpKyeudKgcaoKNEk1KZ0CjVNFAcxSTUqnwO5SRaFJqknpFGicKgpNUk1Kp0DjVFFokmpSOqUPjZVZv0yvYaaiRf3NFE23vhlP1YwW++foNACz0rce5UGB7hXow8F0X4uyZ4AmZfXMlQaNU1WgSapJ6RRonCoKYJZqUjoFdpcqCk1STUqnQONUUWiSalI6BRqnikKTVJPSKX1oHEzJZAC23y2AWelbj/KgQPcK9OFguq9F2TNAk7J65kqDxqkq0CTVpHQKNE4VBTBLNSmdArtLFYUmqSalU6Bxqig0STUpnQKNU0WhSapJ6ZQ+NAYwa7hrfdyAhtMjCwp0qgDsO5UXmqSalE6Bxqmi0CTVpHQKNE4VBTBLNSmdArtLFYUmqSalU6Bxqig0STUpnQKNU0WhSapJ6ZQ+NAYwa7hrfdyAhtMjCwp0qgDsO5UXmqSalE6Bxqmi0CTVpHQKNC6tKMprowDsLlUJmqSalE6Bxqmi0CTVpHQKNE4VhSapJqVT+tAYwKzhrvVxAxpOjywo0KkCsO9UXmiSalI6BRqnikKTVJPSKdC4tKIor40CsLtUJWiSalI6BRqnikKTVJPSKdA4VRSapJqUTulDY3X//n26d+8e3blzh27fvk23bt2i3d1dunnzJu3s7NCNGzfo+vXrdO3aNbp69SpduXKFLl++TJcuXaKLFy/ShQsXzFsy9cXiDxrABmADsAHYAGwANgAbWHwb0J1W3KfFv0+4R7hHsAHYAGwANgAbmJ8NIMKsAXNqw8Q/KDBWBWDf6Z2FJqkmpVOgcaooNEk1KZ0CjVNFsYZZqknpFNhdqig0STUpnQKNU0WhSapJ6RRonCoKTVJNSqf0oTGAWcNd6+MGNJweWVCgUwVg36m80CTVpHQKNE4VhSapJqVToHGqKIBZqknpFNhdqig0STUpnQKNU0WhSapJ6RRonCoKTVJNSqf0oTGAWcNd6+MGNJweWVCgUwVg36m80CTVpHQKNE4VhSapJqVToHGqKIBZqknpFNhdqig0STUpnQKNU0WhSapJ6RRonCoKTVJNSqf0oTGAWcNd6+MGNJweWVCgUwVg36m80CTVpHQKNE4VhSapJqVToHGqKIBZqknpFNhdqig0STUpnQKNU0WhSapJ6RRonCoKTVJNSqf0ofH/B4e+FL11YC3pAAAAAElFTkSuQmCC" alt="image">

It can be a problem of blitter, and we will make reproducer at next week